<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Differential OCR analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</title>
  <meta name="description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Differential OCR analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Differential OCR analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  
  <meta name="twitter:description" content="cis-regulatory chromatin dynamics and networks." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-08-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="peak-annotaion.html"/>
<link rel="next" href="tissuesample-specific-analysis.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">cisDynet</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Primer</a></li>
<li class="chapter" data-level="2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html"><i class="fa fa-check"></i><b>2</b> Getting Started with cisDynet</a>
<ul>
<li class="chapter" data-level="2.1" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#why-use-cisdynet"><i class="fa fa-check"></i><b>2.1</b> Why use cisDynet?</a></li>
<li class="chapter" data-level="2.2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#getting-set-up"><i class="fa fa-check"></i><b>2.2</b> Getting Set Up</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>3</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="3.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#example-data"><i class="fa fa-check"></i><b>3.1</b> Example Data</a></li>
<li class="chapter" data-level="3.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#config-file"><i class="fa fa-check"></i><b>3.2</b> Config File</a></li>
<li class="chapter" data-level="3.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#blacklists"><i class="fa fa-check"></i><b>3.3</b> Blacklists</a></li>
<li class="chapter" data-level="3.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#pipeline-output-folders"><i class="fa fa-check"></i><b>3.4</b> Pipeline Output Folders</a></li>
<li class="chapter" data-level="3.5" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explanation"><i class="fa fa-check"></i><b>3.5</b> Report Explanation</a></li>
<li class="chapter" data-level="3.6" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#troubleshooting"><i class="fa fa-check"></i><b>3.6</b> Troubleshooting</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="quality-control-plot.html"><a href="quality-control-plot.html"><i class="fa fa-check"></i><b>4</b> Quality Control Plot</a>
<ul>
<li class="chapter" data-level="4.1" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plotfragments"><i class="fa fa-check"></i><b>4.1</b> plotFragments</a></li>
<li class="chapter" data-level="4.2" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plottss"><i class="fa fa-check"></i><b>4.2</b> plotTSS</a></li>
<li class="chapter" data-level="4.3" data-path="quality-control-plot.html"><a href="quality-control-plot.html#quantification"><i class="fa fa-check"></i><b>4.3</b> Quantification</a></li>
<li class="chapter" data-level="4.4" data-path="quality-control-plot.html"><a href="quality-control-plot.html#pca"><i class="fa fa-check"></i><b>4.4</b> PCA</a></li>
<li class="chapter" data-level="4.5" data-path="quality-control-plot.html"><a href="quality-control-plot.html#dendrogram"><i class="fa fa-check"></i><b>4.5</b> Dendrogram</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>5</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotvenn2---6-samples"><i class="fa fa-check"></i><b>5.1</b> PlotVenn(2 - 6 samples)</a></li>
<li class="chapter" data-level="5.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotupset"><i class="fa fa-check"></i><b>5.2</b> plotUpSet</a></li>
<li class="chapter" data-level="5.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>5.3</b> Pairwise module examples</a></li>
<li class="chapter" data-level="5.4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#flower-plot"><i class="fa fa-check"></i><b>5.4</b> Flower plot</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>6</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>6.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="6.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#the-distance-of-peak-summit-to-tss"><i class="fa fa-check"></i><b>6.2</b> The distance of Peak summit to TSS</a></li>
<li class="chapter" data-level="6.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#proximal-distal-intragenic-annotaion"><i class="fa fa-check"></i><b>6.3</b> Proximal / Distal / Intragenic Annotaion</a></li>
<li class="chapter" data-level="6.4" data-path="peak-annotaion.html"><a href="peak-annotaion.html#annotation-for-merged-peaks"><i class="fa fa-check"></i><b>6.4</b> Annotation for merged peaks</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html"><i class="fa fa-check"></i><b>7</b> Differential OCR analysis</a>
<ul>
<li class="chapter" data-level="7.1" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups"><i class="fa fa-check"></i><b>7.1</b> Using DEseq2 to identify differentail OCR (Two groups)</a></li>
<li class="chapter" data-level="7.2" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes"><i class="fa fa-check"></i><b>7.2</b> Get the differential peaks target genes</a></li>
<li class="chapter" data-level="7.3" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#plotma"><i class="fa fa-check"></i><b>7.3</b> PlotMA</a></li>
<li class="chapter" data-level="7.4" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples"><i class="fa fa-check"></i><b>7.4</b> Compare the three samples chromatin accessibility (Three samples)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html"><i class="fa fa-check"></i><b>8</b> Tissue(Sample) Specific Analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix"><i class="fa fa-check"></i><b>8.1</b> Calaluate the SPM matrix</a></li>
<li class="chapter" data-level="8.2" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests"><i class="fa fa-check"></i><b>8.2</b> Select the specific peak with your interests</a></li>
<li class="chapter" data-level="8.3" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score"><i class="fa fa-check"></i><b>8.3</b> Select the specifc peaks with JSD score</a></li>
<li class="chapter" data-level="8.4" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues"><i class="fa fa-check"></i><b>8.4</b> Get the Top N specific peaks in every Samples / Tissues</a></li>
<li class="chapter" data-level="8.5" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number"><i class="fa fa-check"></i><b>8.5</b> Get the optimal cluster number</a></li>
<li class="chapter" data-level="8.6" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes"><i class="fa fa-check"></i><b>8.6</b> Get the cluster target genes</a></li>
<li class="chapter" data-level="8.7" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#go-enrichment"><i class="fa fa-check"></i><b>8.7</b> GO Enrichment</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="time-course-data.html"><a href="time-course-data.html"><i class="fa fa-check"></i><b>9</b> Time Course Data</a>
<ul>
<li class="chapter" data-level="9.1" data-path="time-course-data.html"><a href="time-course-data.html#fitting-time-with-atac-seq"><i class="fa fa-check"></i><b>9.1</b> Fitting time with ATAC-seq</a></li>
<li class="chapter" data-level="9.2" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-rna-seq"><i class="fa fa-check"></i><b>9.2</b> Integration with RNA-seq</a></li>
<li class="chapter" data-level="9.3" data-path="time-course-data.html"><a href="time-course-data.html#get-fitting-time-genes"><i class="fa fa-check"></i><b>9.3</b> Get fitting time genes</a></li>
<li class="chapter" data-level="9.4" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-atac-rna-seq"><i class="fa fa-check"></i><b>9.4</b> Integration with ATAC / RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>10</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="10.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#chromvar"><i class="fa fa-check"></i><b>10.1</b> ChromVAR</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="footprint-analysis.html"><a href="footprint-analysis.html"><i class="fa fa-check"></i><b>11</b> Footprint Analysis</a>
<ul>
<li class="chapter" data-level="11.1" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-footprint-score"><i class="fa fa-check"></i><b>11.1</b> Get footprint score</a></li>
<li class="chapter" data-level="11.2" data-path="footprint-analysis.html"><a href="footprint-analysis.html#plot-footprint-score"><i class="fa fa-check"></i><b>11.2</b> Plot footprint score</a></li>
<li class="chapter" data-level="11.3" data-path="footprint-analysis.html"><a href="footprint-analysis.html#compare-footprints"><i class="fa fa-check"></i><b>11.3</b> Compare Footprints</a></li>
<li class="chapter" data-level="11.4" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-differential-footprints"><i class="fa fa-check"></i><b>11.4</b> Get Differential Footprints</a></li>
<li class="chapter" data-level="11.5" data-path="footprint-analysis.html"><a href="footprint-analysis.html#getmotif2gene"><i class="fa fa-check"></i><b>11.5</b> getMotif2Gene</a></li>
<li class="chapter" data-level="11.6" data-path="footprint-analysis.html"><a href="footprint-analysis.html#integration-with-rna-seq-1"><i class="fa fa-check"></i><b>11.6</b> Integration with RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="co-accessibility.html"><a href="co-accessibility.html"><i class="fa fa-check"></i><b>12</b> Co-accessibility</a>
<ul>
<li class="chapter" data-level="12.1" data-path="co-accessibility.html"><a href="co-accessibility.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>12.1</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="12.2" data-path="co-accessibility.html"><a href="co-accessibility.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>12.2</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>13</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="13.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#get-the-peak-target-genes"><i class="fa fa-check"></i><b>13.1</b> Get the peak target genes</a></li>
<li class="chapter" data-level="13.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plot-all-the-significant-peak2gene-links"><i class="fa fa-check"></i><b>13.2</b> Plot all the significant peak2gene links</a></li>
<li class="chapter" data-level="13.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>13.3</b> Link OCR to genes</a></li>
<li class="chapter" data-level="13.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotp2gtracks"><i class="fa fa-check"></i><b>13.4</b> plotP2GTracks</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>14</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="14.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#motif2gene"><i class="fa fa-check"></i><b>14.1</b> Motif2Gene</a></li>
<li class="chapter" data-level="14.2" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#building-regulatory-network-1"><i class="fa fa-check"></i><b>14.2</b> Building regulatory network</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html"><i class="fa fa-check"></i><b>15</b> GWAS Trait SNPs Enrichment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#workflows"><i class="fa fa-check"></i><b>15.1</b> Workflows</a></li>
<li class="chapter" data-level="15.2" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#preprocessing"><i class="fa fa-check"></i><b>15.2</b> Preprocessing</a></li>
<li class="chapter" data-level="15.3" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#do-enrichment"><i class="fa fa-check"></i><b>15.3</b> Do Enrichment</a></li>
<li class="chapter" data-level="15.4" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#plotting"><i class="fa fa-check"></i><b>15.4</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="other-utils.html"><a href="other-utils.html"><i class="fa fa-check"></i><b>16</b> Other Utils</a>
<ul>
<li class="chapter" data-level="16.1" data-path="other-utils.html"><a href="other-utils.html#super-enhancer"><i class="fa fa-check"></i><b>16.1</b> Super Enhancer</a></li>
<li class="chapter" data-level="16.2" data-path="other-utils.html"><a href="other-utils.html#peak-fisher-test"><i class="fa fa-check"></i><b>16.2</b> Peak Fisher Test</a></li>
<li class="chapter" data-level="16.3" data-path="other-utils.html"><a href="other-utils.html#snp2gene"><i class="fa fa-check"></i><b>16.3</b> SNP2Gene</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="session-information.html"><a href="session-information.html"><i class="fa fa-check"></i><b>17</b> Session Information</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="differential-ocr-analysis" class="section level1 hasAnchor" number="7">
<h1><span class="header-section-number">Chapter 7</span> Differential OCR analysis<a href="differential-ocr-analysis.html#differential-ocr-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Differential peak analysis can help us identify differences in gene regulation under different conditions and reveal important regulatory mechanisms in cellular states and biological processes. This section of the analysis is an important step in the search for differential TFBSs and is the basis for performing a functional difference analysis.</p>
<div id="using-deseq2-to-identify-differentail-ocr-two-groups" class="section level2 hasAnchor" number="7.1">
<h2><span class="header-section-number">7.1</span> Using DEseq2 to identify differentail OCR (Two groups)<a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>DESeq2 is an R package for differential expression analysis, but can also be used for differential peak analysis with ATAC-seq. Here we use DEseq2 as an example to identify the difference peaks between the two groups.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="differential-ocr-analysis.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">data</span>(Diff_Analysis_Example)</span>
<span id="cb1-2"><a href="differential-ocr-analysis.html#cb1-2" aria-hidden="true" tabindex="-1"></a>quant_data <span class="ot">&lt;-</span> Diff_Analysis_Example</span>
<span id="cb1-3"><a href="differential-ocr-analysis.html#cb1-3" aria-hidden="true" tabindex="-1"></a>diff_res <span class="ot">&lt;-</span> <span class="fu">getDiffPeak</span>(<span class="at">norm_data =</span> quant_data, <span class="at">condition =</span> <span class="fu">c</span>(<span class="st">&quot;Control&quot;</span>,<span class="st">&quot;Exper&quot;</span>), <span class="at">control =</span> <span class="st">&quot;Control&quot;</span>, <span class="at">experment =</span> <span class="st">&quot;Exper&quot;</span>,<span class="at">rep_N=</span> <span class="dv">2</span>)</span></code></pre></div>
<pre><code>## converting counts to integer mode</code></pre>
<pre><code>## estimating size factors</code></pre>
<pre><code>## estimating dispersions</code></pre>
<pre><code>## gene-wise dispersion estimates: 22 workers</code></pre>
<pre><code>## mean-dispersion relationship</code></pre>
<pre><code>## final dispersion estimates, fitting model and testing: 22 workers</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="differential-ocr-analysis.html#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(diff_res)</span></code></pre></div>
<pre><code>##                     baseMean log2FoldChange     lfcSE       stat    pvalue
## chr1:10016-10607   54.030792      0.5635555 0.5826459  0.9672349 0.3334266
## chr1:237589-237928 11.903993      0.3358489 0.7164559  0.4687643 0.6392381
## chr1:521379-521712  6.946804      0.1798363 0.8110918  0.2217212 0.8245309
## chr1:540496-541113  8.700218     -0.1107456 0.7780795 -0.1423320 0.8868178
## chr1:559303-559603  8.693440     -0.2740060 0.7751323 -0.3534957 0.7237168
## chr1:564421-565042 24.555285      0.2952452 0.6325501  0.4667538 0.6406760
##                         padj  sig
## chr1:10016-10607   0.7779954 None
## chr1:237589-237928 0.9452912 None
## chr1:521379-521712 0.9800009 None
## chr1:540496-541113 0.9845317 None
## chr1:559303-559603 0.9657660 None
## chr1:564421-565042 0.9456528 None</code></pre>
<p>We can use the commonly used volcano diagram to show these differential peaks.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="differential-ocr-analysis.html#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotVolcano</span>(<span class="at">diff_data =</span> diff_res)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plotVolcano-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="get-the-differential-peaks-target-genes" class="section level2 hasAnchor" number="7.2">
<h2><span class="header-section-number">7.2</span> Get the differential peaks target genes<a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Having obtained these differential peaks, we were more interested in knowing which genes’ expression might be affected by these differential peaks. In general, we annotate these differential peaks to the nearest genes. We can run the <strong>getDiffTargetGenes</strong> function to complete this analysis.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="differential-ocr-analysis.html#cb11-1" aria-hidden="true" tabindex="-1"></a>target <span class="ot">&lt;-</span> <span class="fu">getDiffTargetGenes</span>(<span class="at">diff_data=</span>diff_res)</span>
<span id="cb11-2"><a href="differential-ocr-analysis.html#cb11-2" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(target)</span></code></pre></div>
<pre><code>##   chrom     start       end  baseMean log2FoldChange     lfcSE       stat
## 1  chr1 100009953 100010254 20.142897     -0.4939808 0.6575732 -0.7512181
## 2  chr1 100012048 100012826 23.103692     -0.3060881 0.6464743 -0.4734730
## 3  chr1 100017890 100018149  8.223809     -0.8358235 0.8052071 -1.0380231
## 4  chr1 100023076 100023997 22.811175      0.2890909 0.6505642  0.4443696
## 5  chr1  10002486  10002743  8.659492      1.0989500 0.7945523  1.3831059
## 6  chr1  10002897  10003700 55.147809     -0.1827790 0.5806837 -0.3147652
##      pvalue      padj  sig peak_center       TSS gene strand distance
## 1 0.4525214 0.8669849 None   100010103  99929934    1      +   -80169
## 2 0.6358758 0.9436658 None   100012437  99929934    1      +   -82503
## 3 0.2992593 0.7424509 None   100018019  99929934    1      +   -88085
## 4 0.6567754 0.9507410 None   100023536 100111499    1      +    87963
## 5 0.1666324 0.5544505 None    10002614  10003465    1      -     -851
## 6 0.7529400 0.9699603 None    10003298  10003465    1      -     -167</code></pre>
</div>
<div id="plotma" class="section level2 hasAnchor" number="7.3">
<h2><span class="header-section-number">7.3</span> PlotMA<a href="differential-ocr-analysis.html#plotma" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In addition, we can also use the plotMA function to display information about both the difference peaks and their summit to the nearest TSS distance.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="differential-ocr-analysis.html#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotMA</span>(target)</span></code></pre></div>
<pre><code>## Warning: Removed 21 rows containing missing values (`geom_point()`).</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotMA-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="compare-the-three-samples-chromatin-accessibility-three-samples" class="section level2 hasAnchor" number="7.4">
<h2><span class="header-section-number">7.4</span> Compare the three samples chromatin accessibility (Three samples)<a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>For the analysis of differences among the three groups, a common method is a two-by-two comparison. Here, we are inspired by <a href="https://www.science.org/doi/10.1126/science.aar6089">R. H. Ramírez-González et al</a>. For three groups of ATAC-seq data, we can use this triads approach to get the difference peaks at the same time and visualized with a ternary plot. We take the previous quantized matrix and extract the <em>Bulk_B</em>, <em>Mem_B</em>, <em>Naive_B</em> to compare the differences between the three groups.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="differential-ocr-analysis.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;All_quant_data&quot;</span>)</span>
<span id="cb15-2"><a href="differential-ocr-analysis.html#cb15-2" aria-hidden="true" tabindex="-1"></a>quant_data <span class="ot">&lt;-</span> All_quant_data[,<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>]</span>
<span id="cb15-3"><a href="differential-ocr-analysis.html#cb15-3" aria-hidden="true" tabindex="-1"></a><span class="fu">getTriads</span>(<span class="at">group_name=</span><span class="fu">c</span>(<span class="st">&quot;Bulk_B&quot;</span>,<span class="st">&quot;Mem_B&quot;</span>,<span class="st">&quot;Naive_B&quot;</span>), <span class="at">quant_df =</span> quant_data, <span class="at">return_matrix=</span>F)</span></code></pre></div>
<pre><code>## Registered S3 methods overwritten by &#39;ggtern&#39;:
##   method           from   
##   grid.draw.ggplot ggplot2
##   plot.ggplot      ggplot2
##   print.ggplot     ggplot2</code></pre>
<p><img src="bookdown-demo_files/figure-html/getTraids-1.png" width="672" style="display: block; margin: auto;" />
We can categorize them into seven categories and use a boxplot to show if their categorization is accurate.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="differential-ocr-analysis.html#cb17-1" aria-hidden="true" tabindex="-1"></a>triads_res <span class="ot">&lt;-</span> <span class="fu">getTriads</span>(<span class="at">group_name=</span><span class="fu">c</span>(<span class="st">&quot;Bulk_B&quot;</span>,<span class="st">&quot;Mem_B&quot;</span>,<span class="st">&quot;Naive_B&quot;</span>), <span class="at">quant_df =</span> quant_data, <span class="at">return_matrix=</span>T)</span>
<span id="cb17-2"><a href="differential-ocr-analysis.html#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotTriads</span>(<span class="at">triads_data =</span> triads_res)</span></code></pre></div>
<pre><code>## Using group as id variables</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotTraids-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Finally, we can similarly use the getTraidsTargetGenes function to get the target genes for each category of peaks.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="differential-ocr-analysis.html#cb19-1" aria-hidden="true" tabindex="-1"></a>triads_targets <span class="ot">&lt;-</span> <span class="fu">getTriadsTargetGenes</span>(<span class="at">triads_data =</span> triads_res)</span>
<span id="cb19-2"><a href="differential-ocr-analysis.html#cb19-2" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(triads_targets)</span></code></pre></div>
<pre><code>##   chrom Peak_center     Bulk_B     Mem_B    Naive_B    group Peak_start
## 1  chr1    10002672 10.9749479  8.274032 10.8202002 Balanced   10002429
## 2  chr1    10003391 49.4307820 41.862457 46.2388672 Balanced   10002983
## 3  chr1    10010875 25.6538466 23.160242 25.3581749 Balanced   10010299
## 4  chr1   100151436  2.8350983  3.277752  1.8329218 Balanced  100151228
## 5  chr1   100165806  0.6880433  0.615438  0.4909362 Balanced  100165637
## 6  chr1   100194636  0.8866477  1.970254  1.2235983 Balanced  100194551
##    Peak_end       TSS            gene strand distance
## 1  10002916  10003465 ENSG00000162441      -     -793
## 2  10003799  10003465 ENSG00000162441      -      -74
## 3  10011452  10003465 ENSG00000162441      -     7410
## 4 100151645 100163798 ENSG00000223656      +    12362
## 5 100165976 100163798 ENSG00000223656      +    -2008
## 6 100194721 100163798 ENSG00000223656      +   -30838</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="peak-annotaion.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="tissuesample-specific-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/05-Differential_OCR_analysis.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
