<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 11 Footprint Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</title>
  <meta name="description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 11 Footprint Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 11 Footprint Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  
  <meta name="twitter:description" content="cis-regulatory chromatin dynamics and networks." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-08-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="motif-enrichment.html"/>
<link rel="next" href="co-accessibility.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">cisDynet</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Primer</a></li>
<li class="chapter" data-level="2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html"><i class="fa fa-check"></i><b>2</b> Getting Started with cisDynet</a>
<ul>
<li class="chapter" data-level="2.1" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#why-use-cisdynet"><i class="fa fa-check"></i><b>2.1</b> Why use cisDynet?</a></li>
<li class="chapter" data-level="2.2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#getting-set-up"><i class="fa fa-check"></i><b>2.2</b> Getting Set Up</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>3</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="3.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#example-data"><i class="fa fa-check"></i><b>3.1</b> Example Data</a></li>
<li class="chapter" data-level="3.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#config-file"><i class="fa fa-check"></i><b>3.2</b> Config File</a></li>
<li class="chapter" data-level="3.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#blacklists"><i class="fa fa-check"></i><b>3.3</b> Blacklists</a></li>
<li class="chapter" data-level="3.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#pipeline-output-folders"><i class="fa fa-check"></i><b>3.4</b> Pipeline Output Folders</a></li>
<li class="chapter" data-level="3.5" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explanation"><i class="fa fa-check"></i><b>3.5</b> Report Explanation</a></li>
<li class="chapter" data-level="3.6" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#troubleshooting"><i class="fa fa-check"></i><b>3.6</b> Troubleshooting</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="quality-control-plot.html"><a href="quality-control-plot.html"><i class="fa fa-check"></i><b>4</b> Quality Control Plot</a>
<ul>
<li class="chapter" data-level="4.1" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plotfragments"><i class="fa fa-check"></i><b>4.1</b> plotFragments</a></li>
<li class="chapter" data-level="4.2" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plottss"><i class="fa fa-check"></i><b>4.2</b> plotTSS</a></li>
<li class="chapter" data-level="4.3" data-path="quality-control-plot.html"><a href="quality-control-plot.html#quantification"><i class="fa fa-check"></i><b>4.3</b> Quantification</a></li>
<li class="chapter" data-level="4.4" data-path="quality-control-plot.html"><a href="quality-control-plot.html#pca"><i class="fa fa-check"></i><b>4.4</b> PCA</a></li>
<li class="chapter" data-level="4.5" data-path="quality-control-plot.html"><a href="quality-control-plot.html#dendrogram"><i class="fa fa-check"></i><b>4.5</b> Dendrogram</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>5</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotvenn2---6-samples"><i class="fa fa-check"></i><b>5.1</b> PlotVenn(2 - 6 samples)</a></li>
<li class="chapter" data-level="5.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotupset"><i class="fa fa-check"></i><b>5.2</b> plotUpSet</a></li>
<li class="chapter" data-level="5.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>5.3</b> Pairwise module examples</a></li>
<li class="chapter" data-level="5.4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#flower-plot"><i class="fa fa-check"></i><b>5.4</b> Flower plot</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>6</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>6.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="6.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#the-distance-of-peak-summit-to-tss"><i class="fa fa-check"></i><b>6.2</b> The distance of Peak summit to TSS</a></li>
<li class="chapter" data-level="6.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#proximal-distal-intragenic-annotaion"><i class="fa fa-check"></i><b>6.3</b> Proximal / Distal / Intragenic Annotaion</a></li>
<li class="chapter" data-level="6.4" data-path="peak-annotaion.html"><a href="peak-annotaion.html#annotation-for-merged-peaks"><i class="fa fa-check"></i><b>6.4</b> Annotation for merged peaks</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html"><i class="fa fa-check"></i><b>7</b> Differential OCR analysis</a>
<ul>
<li class="chapter" data-level="7.1" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups"><i class="fa fa-check"></i><b>7.1</b> Using DEseq2 to identify differentail OCR (Two groups)</a></li>
<li class="chapter" data-level="7.2" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes"><i class="fa fa-check"></i><b>7.2</b> Get the differential peaks target genes</a></li>
<li class="chapter" data-level="7.3" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#plotma"><i class="fa fa-check"></i><b>7.3</b> PlotMA</a></li>
<li class="chapter" data-level="7.4" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples"><i class="fa fa-check"></i><b>7.4</b> Compare the three samples chromatin accessibility (Three samples)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html"><i class="fa fa-check"></i><b>8</b> Tissue(Sample) Specific Analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix"><i class="fa fa-check"></i><b>8.1</b> Calaluate the SPM matrix</a></li>
<li class="chapter" data-level="8.2" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests"><i class="fa fa-check"></i><b>8.2</b> Select the specific peak with your interests</a></li>
<li class="chapter" data-level="8.3" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score"><i class="fa fa-check"></i><b>8.3</b> Select the specifc peaks with JSD score</a></li>
<li class="chapter" data-level="8.4" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues"><i class="fa fa-check"></i><b>8.4</b> Get the Top N specific peaks in every Samples / Tissues</a></li>
<li class="chapter" data-level="8.5" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number"><i class="fa fa-check"></i><b>8.5</b> Get the optimal cluster number</a></li>
<li class="chapter" data-level="8.6" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes"><i class="fa fa-check"></i><b>8.6</b> Get the cluster target genes</a></li>
<li class="chapter" data-level="8.7" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#go-enrichment"><i class="fa fa-check"></i><b>8.7</b> GO Enrichment</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="time-course-data.html"><a href="time-course-data.html"><i class="fa fa-check"></i><b>9</b> Time Course Data</a>
<ul>
<li class="chapter" data-level="9.1" data-path="time-course-data.html"><a href="time-course-data.html#fitting-time-with-atac-seq"><i class="fa fa-check"></i><b>9.1</b> Fitting time with ATAC-seq</a></li>
<li class="chapter" data-level="9.2" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-rna-seq"><i class="fa fa-check"></i><b>9.2</b> Integration with RNA-seq</a></li>
<li class="chapter" data-level="9.3" data-path="time-course-data.html"><a href="time-course-data.html#get-fitting-time-genes"><i class="fa fa-check"></i><b>9.3</b> Get fitting time genes</a></li>
<li class="chapter" data-level="9.4" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-atac-rna-seq"><i class="fa fa-check"></i><b>9.4</b> Integration with ATAC / RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>10</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="10.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#chromvar"><i class="fa fa-check"></i><b>10.1</b> ChromVAR</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="footprint-analysis.html"><a href="footprint-analysis.html"><i class="fa fa-check"></i><b>11</b> Footprint Analysis</a>
<ul>
<li class="chapter" data-level="11.1" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-footprint-score"><i class="fa fa-check"></i><b>11.1</b> Get footprint score</a></li>
<li class="chapter" data-level="11.2" data-path="footprint-analysis.html"><a href="footprint-analysis.html#plot-footprint-score"><i class="fa fa-check"></i><b>11.2</b> Plot footprint score</a></li>
<li class="chapter" data-level="11.3" data-path="footprint-analysis.html"><a href="footprint-analysis.html#compare-footprints"><i class="fa fa-check"></i><b>11.3</b> Compare Footprints</a></li>
<li class="chapter" data-level="11.4" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-differential-footprints"><i class="fa fa-check"></i><b>11.4</b> Get Differential Footprints</a></li>
<li class="chapter" data-level="11.5" data-path="footprint-analysis.html"><a href="footprint-analysis.html#getmotif2gene"><i class="fa fa-check"></i><b>11.5</b> getMotif2Gene</a></li>
<li class="chapter" data-level="11.6" data-path="footprint-analysis.html"><a href="footprint-analysis.html#integration-with-rna-seq-1"><i class="fa fa-check"></i><b>11.6</b> Integration with RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="co-accessibility.html"><a href="co-accessibility.html"><i class="fa fa-check"></i><b>12</b> Co-accessibility</a>
<ul>
<li class="chapter" data-level="12.1" data-path="co-accessibility.html"><a href="co-accessibility.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>12.1</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="12.2" data-path="co-accessibility.html"><a href="co-accessibility.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>12.2</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>13</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="13.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#get-the-peak-target-genes"><i class="fa fa-check"></i><b>13.1</b> Get the peak target genes</a></li>
<li class="chapter" data-level="13.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plot-all-the-significant-peak2gene-links"><i class="fa fa-check"></i><b>13.2</b> Plot all the significant peak2gene links</a></li>
<li class="chapter" data-level="13.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>13.3</b> Link OCR to genes</a></li>
<li class="chapter" data-level="13.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotp2gtracks"><i class="fa fa-check"></i><b>13.4</b> plotP2GTracks</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>14</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="14.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#motif2gene"><i class="fa fa-check"></i><b>14.1</b> Motif2Gene</a></li>
<li class="chapter" data-level="14.2" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#building-regulatory-network-1"><i class="fa fa-check"></i><b>14.2</b> Building regulatory network</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html"><i class="fa fa-check"></i><b>15</b> GWAS Trait SNPs Enrichment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#workflows"><i class="fa fa-check"></i><b>15.1</b> Workflows</a></li>
<li class="chapter" data-level="15.2" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#preprocessing"><i class="fa fa-check"></i><b>15.2</b> Preprocessing</a></li>
<li class="chapter" data-level="15.3" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#do-enrichment"><i class="fa fa-check"></i><b>15.3</b> Do Enrichment</a></li>
<li class="chapter" data-level="15.4" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#plotting"><i class="fa fa-check"></i><b>15.4</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="other-utils.html"><a href="other-utils.html"><i class="fa fa-check"></i><b>16</b> Other Utils</a>
<ul>
<li class="chapter" data-level="16.1" data-path="other-utils.html"><a href="other-utils.html#super-enhancer"><i class="fa fa-check"></i><b>16.1</b> Super Enhancer</a></li>
<li class="chapter" data-level="16.2" data-path="other-utils.html"><a href="other-utils.html#peak-fisher-test"><i class="fa fa-check"></i><b>16.2</b> Peak Fisher Test</a></li>
<li class="chapter" data-level="16.3" data-path="other-utils.html"><a href="other-utils.html#snp2gene"><i class="fa fa-check"></i><b>16.3</b> SNP2Gene</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="session-information.html"><a href="session-information.html"><i class="fa fa-check"></i><b>17</b> Session Information</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="footprint-analysis" class="section level1 hasAnchor" number="11">
<h1><span class="header-section-number">Chapter 11</span> Footprint Analysis<a href="footprint-analysis.html#footprint-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>When a TF is bound to a DNA region, the transposase is unable to insert adapters at that location, creating a “footprint” in the sequencing data. This footprint appears as a region where the ATAC-seq reads are reduced or absent compared to the surrounding chromatin.</p>
<p>Nowadays, there are already many tools for identifying footprints, such as <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1642-2">HINT-ATAC</a>, <a href="https://www.nature.com/articles/s41467-020-18035-1">TOBIAS</a>, <a href="https://academic.oup.com/nar/article/42/17/11272/2902561">Wellington</a> and so on. Here we use TOBIAS to analyze the footprint.</p>
<p>When we do footprint analysis, we need to add the necessary file paths first with <strong>addFootprint</strong>.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="footprint-analysis.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">addFootprint</span>(<span class="at">corrected_signal =</span> <span class="st">&quot;F:/cisDynet/example/ATACcorrect/&quot;</span>,<span class="at">bindetect_result =</span> <span class="st">&quot;F:/cisDynet/example/BINDetect/&quot;</span>)</span></code></pre></div>
<div id="get-footprint-score" class="section level2 hasAnchor" number="11.1">
<h2><span class="header-section-number">11.1</span> Get footprint score<a href="footprint-analysis.html#get-footprint-score" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We can get the activity of each TF in each sample by <strong>getFootprintScore</strong>.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="footprint-analysis.html#cb2-1" aria-hidden="true" tabindex="-1"></a>ftScore <span class="ot">&lt;-</span> <span class="fu">getFootprintScore</span>()</span>
<span id="cb2-2"><a href="footprint-analysis.html#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(ftScore)</span></code></pre></div>
<pre><code>##         output_prefix  Bulk_B CD8pos_T Central_memory_CD8pos_T
## 1       Arnt_MA0004.1 0.57220  0.57560                 0.53371
## 2    AhrArnt_MA0006.1 0.55047  0.55900                 0.52595
## 3 Ddit3Cebpa_MA0019.1 0.27880  0.27388                 0.26135
## 4      Mecom_MA0029.1 0.18764  0.19913                 0.19897
## 5      FOXF2_MA0030.1 0.22045  0.23356                 0.22986
## 6      FOXD1_MA0031.1 0.22419  0.24244                 0.23578
##   Effector_CD4pos_T Effector_memory_CD8pos_T Follicular_T_Helper Gamma_delta_T
## 1           0.57249                  0.54929             0.56040       0.50959
## 2           0.56402                  0.53134             0.55259       0.49953
## 3           0.25701                  0.27321             0.27869       0.24428
## 4           0.18634                  0.20657             0.20197       0.17834
## 5           0.21966                  0.24062             0.23887       0.20695
## 6           0.22659                  0.24658             0.24859       0.20993
##   Immature_NK Mature_NK   Mem_B Memory_Teffs Memory_Tregs Monocytes Naive_B
## 1     0.51436   0.56104 0.56374      0.52231      0.51496   0.58107 0.56645
## 2     0.49235   0.52990 0.53876      0.51951      0.49828   0.54544 0.53619
## 3     0.24540   0.26540 0.28018      0.22820      0.25163   0.33251 0.26496
## 4     0.17999   0.18981 0.18788      0.16242      0.18863   0.17802 0.18235
## 5     0.20160   0.21726 0.22333      0.19252      0.22295   0.20437 0.21663
## 6     0.20290   0.22179 0.22742      0.19777      0.22851   0.20863 0.22038
##   Naive_CD8_T Naive_Teffs    pDCs Plasmablasts Regulatory_T Th1_precursors
## 1     0.56634     0.56474 0.48592      0.55754      0.51623        0.55022
## 2     0.56175     0.55150 0.46297      0.54157      0.50802        0.54467
## 3     0.29591     0.24658 0.29228      0.27541      0.26275        0.24858
## 4     0.20266     0.16727 0.20202      0.17425      0.19060        0.17656
## 5     0.24787     0.20273 0.22289      0.20567      0.23376        0.20972
## 6     0.26254     0.21163 0.23235      0.20926      0.24104        0.21548</code></pre>
</div>
<div id="plot-footprint-score" class="section level2 hasAnchor" number="11.2">
<h2><span class="header-section-number">11.2</span> Plot footprint score<a href="footprint-analysis.html#plot-footprint-score" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We can visualize the distribution of TF footprint scores in all samples as a way to find differences in TF activity between samples.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="footprint-analysis.html#cb4-1" aria-hidden="true" tabindex="-1"></a>labels <span class="ot">&lt;-</span> <span class="fu">sample</span>(ftScore<span class="sc">$</span>output_prefix, <span class="dv">20</span>) </span>
<span id="cb4-2"><a href="footprint-analysis.html#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotFootprintScore</span>(<span class="at">score_result =</span> ftScore, <span class="at">cluster_N =</span> <span class="dv">6</span>,<span class="at">label_motif =</span> labels)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plotFootprintScore-1.png" width="576" style="display: block; margin: auto;" /></p>
</div>
<div id="compare-footprints" class="section level2 hasAnchor" number="11.3">
<h2><span class="header-section-number">11.3</span> Compare Footprints<a href="footprint-analysis.html#compare-footprints" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>You can use <strong>compareFootprints</strong> to compare the differences in certain samples by comparing the distribution of Tn5 cuts around the binding site on the whole genome for a given TF.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="footprint-analysis.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">compareFootprints</span>(<span class="at">samples =</span> <span class="fu">c</span>(<span class="st">&quot;Bulk_B&quot;</span>,<span class="st">&quot;pDCs&quot;</span>),<span class="at">motif =</span> <span class="st">&quot;ATF2_MA1632.1&quot;</span>,<span class="at">N_cores =</span> <span class="dv">1</span>)</span></code></pre></div>
<pre><code>## 2023-10-23 15:37:53 Reading footprint...</code></pre>
<pre><code>## 2023-10-23 15:37:53 Reading bigwig...</code></pre>
<pre><code>## 2023-10-23 15:38:39 Normalize to matrix...</code></pre>
<pre><code>## Warning: Smoothing are failed for 287 rows because there are very few signals
## overlapped to them. Please use `failed_rows(mat)` to get the index of
## failed rows and consider to remove them.</code></pre>
<pre><code>## No id variables; using all as measure variables</code></pre>
<pre><code>## 2023-10-23 15:39:44 Reading footprint...</code></pre>
<pre><code>## 2023-10-23 15:39:44 Reading bigwig...</code></pre>
<pre><code>## 2023-10-23 15:40:24 Normalize to matrix...</code></pre>
<pre><code>## Warning: Smoothing are failed for 1303 rows because there are very few signals
## overlapped to them. Please use `failed_rows(mat)` to get the index of
## failed rows and consider to remove them.</code></pre>
<pre><code>## No id variables; using all as measure variables</code></pre>
<p><img src="bookdown-demo_files/figure-html/compareFootprints-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="get-differential-footprints" class="section level2 hasAnchor" number="11.4">
<h2><span class="header-section-number">11.4</span> Get Differential Footprints<a href="footprint-analysis.html#get-differential-footprints" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In addition, you can also use the plotDiffFootprint function to get the activity difference of those TFs that are more different for a certain two samples. Your can plot with <em>Volcano</em>, <em>Bar</em>, or <em>Lollipop</em> plot.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="footprint-analysis.html#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotDiffFootprint</span>(<span class="at">samples =</span> <span class="fu">c</span>(<span class="st">&quot;Bulk_B&quot;</span>,<span class="st">&quot;pDCs&quot;</span>), <span class="at">plot_type =</span> <span class="st">&quot;Lollipop&quot;</span>,<span class="at">scale =</span> <span class="dv">4</span>)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plotDiffFootprint-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="getmotif2gene" class="section level2 hasAnchor" number="11.5">
<h2><span class="header-section-number">11.5</span> getMotif2Gene<a href="footprint-analysis.html#getmotif2gene" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In some cases, we may be interested in correlating the activity of a TF with its own expression in order to obtain more reliable and important TFs.</p>
<p>For the following analysis, you need to first prepare a <em>tf_family.txt</em> file with the following contents:</p>
<pre><code>##    Symbol         Ensembl   Family
## 1  ZBTB8B ENSG00000273274     ZBTB
## 2    GSX2 ENSG00000180613 Homeobox
## 3    TBX2 ENSG00000121068    T-box
## 4    PAX8 ENSG00000125618      PAX
## 5 CREB3L1 ENSG00000157613  TF_bZIP
## 6  NKX6-1 ENSG00000163623 Homeobox</code></pre>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="footprint-analysis.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(org.Hs.eg.db)</span></code></pre></div>
<p>We can then use the <strong>getMotif2Gene</strong> function to match the TF with the gene ID and family information.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="footprint-analysis.html#cb19-1" aria-hidden="true" tabindex="-1"></a>m2g <span class="ot">&lt;-</span> <span class="fu">getMotif2Gene</span>(<span class="at">motif =</span> <span class="st">&quot;F:/cisDynet/example/jaspar.moitf.txt&quot;</span>,<span class="at">orgdb =</span> org.Hs.eg.db, <span class="at">tf_family =</span> <span class="st">&quot;F:/cisDynet/example/tf_family.txt&quot;</span>)</span></code></pre></div>
<pre><code>## &#39;select()&#39; returned 1:many mapping between keys and columns</code></pre>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="footprint-analysis.html#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(m2g)</span></code></pre></div>
<pre><code>##           ENSEMBL Motif_ID Motif_Name Gene_Name   Family
## 1 ENSG00000001167 MA0060.3       NFYA      NFYA    NF-YA
## 2 ENSG00000005102 MA0661.1      MEOX1     MEOX1 Homeobox
## 3 ENSG00000005513 MA0868.2       SOX8      SOX8      HMG
## 4 ENSG00000006194 MA0528.2     ZNF263    ZNF263  zf-C2H2
## 5 ENSG00000006377 MA0882.1       DLX6      DLX6 Homeobox
## 6 ENSG00000006468 MA0761.2       ETV1      ETV1      ETS</code></pre>
</div>
<div id="integration-with-rna-seq-1" class="section level2 hasAnchor" number="11.6">
<h2><span class="header-section-number">11.6</span> Integration with RNA-seq<a href="footprint-analysis.html#integration-with-rna-seq-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Finally, we can present the correlation between TF footprint activity and its own expression as a scatter plot.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="footprint-analysis.html#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotFootprintScoreExp</span>(<span class="at">motif2gene =</span> m2g, <span class="at">gene_exp =</span> <span class="st">&quot;F:/cisDynet/example/RSEM_TPM_Results.txt&quot;</span>, <span class="at">footprint_score =</span> ftScore)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plotFootprintScoreExp-1.png" width="960" /></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="motif-enrichment.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="co-accessibility.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/09-Footprinting_with_CAT.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
