<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Snakemake Pipeline | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</title>
  <meta name="description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Snakemake Pipeline | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Snakemake Pipeline | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  
  <meta name="twitter:description" content="cis-regulatory chromatin dynamics and networks." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-08-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="getting-started-with-cisdynet.html"/>
<link rel="next" href="quality-control-plot.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">cisDynet</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Primer</a></li>
<li class="chapter" data-level="2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html"><i class="fa fa-check"></i><b>2</b> Getting Started with cisDynet</a>
<ul>
<li class="chapter" data-level="2.1" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#why-use-cisdynet"><i class="fa fa-check"></i><b>2.1</b> Why use cisDynet?</a></li>
<li class="chapter" data-level="2.2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#getting-set-up"><i class="fa fa-check"></i><b>2.2</b> Getting Set Up</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>3</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="3.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#example-data"><i class="fa fa-check"></i><b>3.1</b> Example Data</a></li>
<li class="chapter" data-level="3.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#config-file"><i class="fa fa-check"></i><b>3.2</b> Config File</a></li>
<li class="chapter" data-level="3.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#blacklists"><i class="fa fa-check"></i><b>3.3</b> Blacklists</a></li>
<li class="chapter" data-level="3.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#pipeline-output-folders"><i class="fa fa-check"></i><b>3.4</b> Pipeline Output Folders</a></li>
<li class="chapter" data-level="3.5" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explanation"><i class="fa fa-check"></i><b>3.5</b> Report Explanation</a></li>
<li class="chapter" data-level="3.6" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#troubleshooting"><i class="fa fa-check"></i><b>3.6</b> Troubleshooting</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="quality-control-plot.html"><a href="quality-control-plot.html"><i class="fa fa-check"></i><b>4</b> Quality Control Plot</a>
<ul>
<li class="chapter" data-level="4.1" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plotfragments"><i class="fa fa-check"></i><b>4.1</b> plotFragments</a></li>
<li class="chapter" data-level="4.2" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plottss"><i class="fa fa-check"></i><b>4.2</b> plotTSS</a></li>
<li class="chapter" data-level="4.3" data-path="quality-control-plot.html"><a href="quality-control-plot.html#quantification"><i class="fa fa-check"></i><b>4.3</b> Quantification</a></li>
<li class="chapter" data-level="4.4" data-path="quality-control-plot.html"><a href="quality-control-plot.html#pca"><i class="fa fa-check"></i><b>4.4</b> PCA</a></li>
<li class="chapter" data-level="4.5" data-path="quality-control-plot.html"><a href="quality-control-plot.html#dendrogram"><i class="fa fa-check"></i><b>4.5</b> Dendrogram</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>5</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotvenn2---6-samples"><i class="fa fa-check"></i><b>5.1</b> PlotVenn(2 - 6 samples)</a></li>
<li class="chapter" data-level="5.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotupset"><i class="fa fa-check"></i><b>5.2</b> plotUpSet</a></li>
<li class="chapter" data-level="5.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>5.3</b> Pairwise module examples</a></li>
<li class="chapter" data-level="5.4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#flower-plot"><i class="fa fa-check"></i><b>5.4</b> Flower plot</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>6</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>6.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="6.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#the-distance-of-peak-summit-to-tss"><i class="fa fa-check"></i><b>6.2</b> The distance of Peak summit to TSS</a></li>
<li class="chapter" data-level="6.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#proximal-distal-intragenic-annotaion"><i class="fa fa-check"></i><b>6.3</b> Proximal / Distal / Intragenic Annotaion</a></li>
<li class="chapter" data-level="6.4" data-path="peak-annotaion.html"><a href="peak-annotaion.html#annotation-for-merged-peaks"><i class="fa fa-check"></i><b>6.4</b> Annotation for merged peaks</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html"><i class="fa fa-check"></i><b>7</b> Differential OCR analysis</a>
<ul>
<li class="chapter" data-level="7.1" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups"><i class="fa fa-check"></i><b>7.1</b> Using DEseq2 to identify differentail OCR (Two groups)</a></li>
<li class="chapter" data-level="7.2" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes"><i class="fa fa-check"></i><b>7.2</b> Get the differential peaks target genes</a></li>
<li class="chapter" data-level="7.3" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#plotma"><i class="fa fa-check"></i><b>7.3</b> PlotMA</a></li>
<li class="chapter" data-level="7.4" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples"><i class="fa fa-check"></i><b>7.4</b> Compare the three samples chromatin accessibility (Three samples)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html"><i class="fa fa-check"></i><b>8</b> Tissue(Sample) Specific Analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix"><i class="fa fa-check"></i><b>8.1</b> Calaluate the SPM matrix</a></li>
<li class="chapter" data-level="8.2" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests"><i class="fa fa-check"></i><b>8.2</b> Select the specific peak with your interests</a></li>
<li class="chapter" data-level="8.3" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score"><i class="fa fa-check"></i><b>8.3</b> Select the specifc peaks with JSD score</a></li>
<li class="chapter" data-level="8.4" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues"><i class="fa fa-check"></i><b>8.4</b> Get the Top N specific peaks in every Samples / Tissues</a></li>
<li class="chapter" data-level="8.5" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number"><i class="fa fa-check"></i><b>8.5</b> Get the optimal cluster number</a></li>
<li class="chapter" data-level="8.6" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes"><i class="fa fa-check"></i><b>8.6</b> Get the cluster target genes</a></li>
<li class="chapter" data-level="8.7" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#go-enrichment"><i class="fa fa-check"></i><b>8.7</b> GO Enrichment</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="time-course-data.html"><a href="time-course-data.html"><i class="fa fa-check"></i><b>9</b> Time Course Data</a>
<ul>
<li class="chapter" data-level="9.1" data-path="time-course-data.html"><a href="time-course-data.html#fitting-time-with-atac-seq"><i class="fa fa-check"></i><b>9.1</b> Fitting time with ATAC-seq</a></li>
<li class="chapter" data-level="9.2" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-rna-seq"><i class="fa fa-check"></i><b>9.2</b> Integration with RNA-seq</a></li>
<li class="chapter" data-level="9.3" data-path="time-course-data.html"><a href="time-course-data.html#get-fitting-time-genes"><i class="fa fa-check"></i><b>9.3</b> Get fitting time genes</a></li>
<li class="chapter" data-level="9.4" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-atac-rna-seq"><i class="fa fa-check"></i><b>9.4</b> Integration with ATAC / RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>10</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="10.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#chromvar"><i class="fa fa-check"></i><b>10.1</b> ChromVAR</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="footprint-analysis.html"><a href="footprint-analysis.html"><i class="fa fa-check"></i><b>11</b> Footprint Analysis</a>
<ul>
<li class="chapter" data-level="11.1" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-footprint-score"><i class="fa fa-check"></i><b>11.1</b> Get footprint score</a></li>
<li class="chapter" data-level="11.2" data-path="footprint-analysis.html"><a href="footprint-analysis.html#plot-footprint-score"><i class="fa fa-check"></i><b>11.2</b> Plot footprint score</a></li>
<li class="chapter" data-level="11.3" data-path="footprint-analysis.html"><a href="footprint-analysis.html#compare-footprints"><i class="fa fa-check"></i><b>11.3</b> Compare Footprints</a></li>
<li class="chapter" data-level="11.4" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-differential-footprints"><i class="fa fa-check"></i><b>11.4</b> Get Differential Footprints</a></li>
<li class="chapter" data-level="11.5" data-path="footprint-analysis.html"><a href="footprint-analysis.html#getmotif2gene"><i class="fa fa-check"></i><b>11.5</b> getMotif2Gene</a></li>
<li class="chapter" data-level="11.6" data-path="footprint-analysis.html"><a href="footprint-analysis.html#integration-with-rna-seq-1"><i class="fa fa-check"></i><b>11.6</b> Integration with RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="co-accessibility.html"><a href="co-accessibility.html"><i class="fa fa-check"></i><b>12</b> Co-accessibility</a>
<ul>
<li class="chapter" data-level="12.1" data-path="co-accessibility.html"><a href="co-accessibility.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>12.1</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="12.2" data-path="co-accessibility.html"><a href="co-accessibility.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>12.2</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>13</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="13.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#get-the-peak-target-genes"><i class="fa fa-check"></i><b>13.1</b> Get the peak target genes</a></li>
<li class="chapter" data-level="13.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plot-all-the-significant-peak2gene-links"><i class="fa fa-check"></i><b>13.2</b> Plot all the significant peak2gene links</a></li>
<li class="chapter" data-level="13.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>13.3</b> Link OCR to genes</a></li>
<li class="chapter" data-level="13.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotp2gtracks"><i class="fa fa-check"></i><b>13.4</b> plotP2GTracks</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>14</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="14.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#motif2gene"><i class="fa fa-check"></i><b>14.1</b> Motif2Gene</a></li>
<li class="chapter" data-level="14.2" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#building-regulatory-network-1"><i class="fa fa-check"></i><b>14.2</b> Building regulatory network</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html"><i class="fa fa-check"></i><b>15</b> GWAS Trait SNPs Enrichment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#workflows"><i class="fa fa-check"></i><b>15.1</b> Workflows</a></li>
<li class="chapter" data-level="15.2" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#preprocessing"><i class="fa fa-check"></i><b>15.2</b> Preprocessing</a></li>
<li class="chapter" data-level="15.3" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#do-enrichment"><i class="fa fa-check"></i><b>15.3</b> Do Enrichment</a></li>
<li class="chapter" data-level="15.4" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#plotting"><i class="fa fa-check"></i><b>15.4</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="other-utils.html"><a href="other-utils.html"><i class="fa fa-check"></i><b>16</b> Other Utils</a>
<ul>
<li class="chapter" data-level="16.1" data-path="other-utils.html"><a href="other-utils.html#super-enhancer"><i class="fa fa-check"></i><b>16.1</b> Super Enhancer</a></li>
<li class="chapter" data-level="16.2" data-path="other-utils.html"><a href="other-utils.html#peak-fisher-test"><i class="fa fa-check"></i><b>16.2</b> Peak Fisher Test</a></li>
<li class="chapter" data-level="16.3" data-path="other-utils.html"><a href="other-utils.html#snp2gene"><i class="fa fa-check"></i><b>16.3</b> SNP2Gene</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="session-information.html"><a href="session-information.html"><i class="fa fa-check"></i><b>17</b> Session Information</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="snakemake-pipeline" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Snakemake Pipeline<a href="snakemake-pipeline.html#snakemake-pipeline" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>The pipeline code is available <a href="https://github.com/tzhu-bio/cisDynet_snakemake">here</a>.
Please refer to this to configure the appropriate working environment.</p>
<div id="example-data" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> Example Data<a href="snakemake-pipeline.html#example-data" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Here we will take the ATAC-seq / RNA-seq data from this <a href="https://www.nature.com/articles/s41588-019-0505-9">publication</a> as an example to demonstrate our snakemake process and downstream analysis with <strong>cisDynet</strong> package.</p>
<p>ATAC-seq and RNA-seq data for several immune cell of the human are included here:</p>
<table>
<thead>
<tr class="header">
<th align="center">ID</th>
<th align="center">Cells</th>
<th align="center">ATAC-seq</th>
<th align="center">RNA-seq</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="center">Bulk_B</td>
<td align="center">SRR7650729</td>
<td align="center">SRR7647654</td>
</tr>
<tr class="even">
<td align="center">2</td>
<td align="center">Mem_B</td>
<td align="center">SRR7650731</td>
<td align="center">SRR7647656</td>
</tr>
<tr class="odd">
<td align="center">3</td>
<td align="center">Naive_B</td>
<td align="center">SRR7650733</td>
<td align="center">SRR7647659</td>
</tr>
<tr class="even">
<td align="center">4</td>
<td align="center">Plasmablasts</td>
<td align="center">SRR7650735</td>
<td align="center">SRR7647660</td>
</tr>
<tr class="odd">
<td align="center">5</td>
<td align="center">CD8pos_T</td>
<td align="center">SRR7650736</td>
<td align="center">SRR11007061</td>
</tr>
<tr class="even">
<td align="center">6</td>
<td align="center">Central_memory_CD8pos_T</td>
<td align="center">SRR7650738</td>
<td align="center">SRR11007063</td>
</tr>
<tr class="odd">
<td align="center">7</td>
<td align="center">Effector_memory_CD8pos_T</td>
<td align="center">SRR7650740</td>
<td align="center">SRR11007065</td>
</tr>
<tr class="even">
<td align="center">8</td>
<td align="center">Naive_CD8_T</td>
<td align="center">SRR7650742</td>
<td align="center">SRR11007067</td>
</tr>
<tr class="odd">
<td align="center">9</td>
<td align="center">Gamma_delta_T</td>
<td align="center">SRR7650744</td>
<td align="center">SRR11007069</td>
</tr>
<tr class="even">
<td align="center">10</td>
<td align="center">Effector_CD4pos_T</td>
<td align="center">SRR7650745</td>
<td align="center">SRR11007071</td>
</tr>
<tr class="odd">
<td align="center">11</td>
<td align="center">Follicular_T_Helper</td>
<td align="center">SRR7650747</td>
<td align="center">SRR11007073</td>
</tr>
<tr class="even">
<td align="center">12</td>
<td align="center">Memory_Teffs</td>
<td align="center">SRR7650749</td>
<td align="center">SRR11007075</td>
</tr>
<tr class="odd">
<td align="center">13</td>
<td align="center">Memory_Tregs</td>
<td align="center">SRR7650751</td>
<td align="center">SRR11007077</td>
</tr>
<tr class="even">
<td align="center">14</td>
<td align="center">Naive_Teffs</td>
<td align="center">SRR7650753</td>
<td align="center">SRR11007079</td>
</tr>
<tr class="odd">
<td align="center">15</td>
<td align="center">Regulatory_T</td>
<td align="center">SRR7650755</td>
<td align="center">SRR11007083</td>
</tr>
<tr class="even">
<td align="center">16</td>
<td align="center">Th1_precursors</td>
<td align="center">SRR7650757</td>
<td align="center">SRR11007085</td>
</tr>
<tr class="odd">
<td align="center">17</td>
<td align="center">Immature_NK</td>
<td align="center">SRR7650763</td>
<td align="center">SRR11007193</td>
</tr>
<tr class="even">
<td align="center">18</td>
<td align="center">Mature_NK</td>
<td align="center">SRR7650764</td>
<td align="center">SRR11007090</td>
</tr>
<tr class="odd">
<td align="center">19</td>
<td align="center">Monocytes</td>
<td align="center">SRR7650767</td>
<td align="center">SRR11007092</td>
</tr>
<tr class="even">
<td align="center">20</td>
<td align="center">pDCs</td>
<td align="center">SRR7650770</td>
<td align="center">SRR11007095</td>
</tr>
</tbody>
</table>
</div>
<div id="config-file" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Config File<a href="snakemake-pipeline.html#config-file" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>All you need to do is prepare a <strong>config.yaml</strong> file in the following format.</p>
<pre><code>sample:  
  Bulk_B:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650729_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650729_2.fastq.gz
  Mem_B:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650731_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650731_2.fastq.gz
  Naive_B:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650733_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650733_2.fastq.gz
  Plasmablasts:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650735_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650735_2.fastq.gz
  CD8pos_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650736_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650736_2.fastq.gz
  Central_memory_CD8pos_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650738_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650738_2.fastq.gz
  Effector_memory_CD8pos_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650740_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650740_2.fastq.gz
  Naive_CD8_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650742_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650742_2.fastq.gz
  Gamma_delta_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650744_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650744_2.fastq.gz
  Effector_CD4pos_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650745_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650745_2.fastq.gz
  Follicular_T_Helper:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650747_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650747_2.fastq.gz
  Memory_Teffs:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650749_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650749_2.fastq.gz
  Memory_Tregs:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650751_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650751_2.fastq.gz
  Naive_Teffs:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650753_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650753_2.fastq.gz
  Regulatory_T:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650755_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650755_2.fastq.gz
  Th1_precursors:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650757_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650757_2.fastq.gz
  Immature_NK:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650763_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650763_2.fastq.gz
  Mature_NK:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650764_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650764_2.fastq.gz
  Monocytes:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650767_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650767_2.fastq.gz
  pDCs:
    fwd: /mnt/public3/cat/data/ATAC/fq/SRR7650770_1.fastq.gz
    rev: /mnt/public3/cat/data/ATAC/fq/SRR7650770_2.fastq.gz


## Pipeline working derectory.    
workdir: /mnt/public3/cat/data/ATAC/

## BWA indexed genome fasta file.
fasta: /public/workspace/zhutao/pipeline/genome/hg19/bwa_clean_index/hg19_clean.fa

## The gff file.
gff3: /public/workspace/zhutao/pipeline/genome/hg19/gencode.v19.annotation.gff3

## Effective genome size for MACS2 peeak calling.
effect_genome_size: 2864785220

## Motif file with PWM format.
motif: /mnt/public3/cat/data/ATAC/jaspar.moitf.txt

## Sequence_adapters.
adapters: /public/workspace/zhutao/miniconda3/pkgs/trimmomatic-0.39-1/share/trimmomatic-0.39-1/adapters/NexteraPE-PE.fa

## mapping quality tags.
mapping_quality: 30

## Organelle name (For plants, a comma separates mitochondria and chloroplasts)
organell_chr: chrX

## Peak calling.
extsize: 150
shift: 75

## convert bam to bigwig.
smoothLength: 20
binSize: 5
threads: 5

## Remove blacklist from the peaks.
blacklist: NA
overlap_ratio: 0.3  ## Requiring the overlap region being at least xx% of peak as a blacklist.

## The software path.
software:
  atacqc: &quot;/mnt/public3/cat/data/ATAC/atacqc&quot;
  trimmomatic: &quot;~/miniconda3/bin/trimmomatic&quot;
  bwa: &quot;~/miniconda3/bin/bwa&quot;
  bedtools: &quot;/bioapps/rhel7/bedtools-2.30.0/bin/bedtools&quot;
  samtools: &quot;~/miniconda3/bin/samtools&quot;
  macs2: &quot;~/miniconda3/bin/macs2&quot;
  bamCoverage: &quot;~/miniconda3/bin/bamCoverage&quot;
  fadix: &quot;~/miniconda3/bin/faidx&quot;
  multiqc: &quot;~/miniconda3/bin/multiqc&quot;
  TOBIAS: &quot;~/miniconda3/envs/tobias/bin/TOBIAS&quot;
</code></pre>
</div>
<div id="blacklists" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Blacklists<a href="snakemake-pipeline.html#blacklists" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In the context of epigenomic analysis, “blacklisted regions” typically refer to specific areas on the genome that often include repetitive sequences, transposons, genes encoding non-epigenetic modifying enzymes, etc. These regions can potentially interfere with the accuracy and interpretability of epigenomic studies. In mammals, such as humans and mice, the ENCODE programme has obtained specific <a href="https://github.com/Boyle-Lab/Blacklist">blacklists</a> by inferring and hand-correcting. This same problem exists in plant epigenomics, but there is no specific blacklist for researchers to use. Here we used the <a href="https://academic.oup.com/plcell/article/34/12/4795/6705244#383964580">Greenscreen</a> software here, combined with data collected by our <a href="https://www.nature.com/articles/s41467-022-30770-1">ChIP-Hub</a>, to infer genome blacklists for Arabidopsis, rice, maize, tomato, and soybean.</p>
<p>The corresponding blacklists can be downloaded via the links below.</p>
<table style="width:100%;">
<colgroup>
<col width="16%" />
<col width="16%" />
<col width="16%" />
<col width="16%" />
<col width="16%" />
<col width="16%" />
</colgroup>
<thead>
<tr class="header">
<th>Plant species</th>
<th>Tax ID</th>
<th>Common name</th>
<th>Genome release version</th>
<th>Genome size (in Mb)</th>
<th># BAMs used</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://github.com/tzhu-bio/cisDynet_snakemake/blob/main/blacklist/arabidopsis_thaliana_Greenscreen_Freq_200_blacklist.bed">Arabidopsis thaliana</a></td>
<td>3702</td>
<td>Mouse-ear cress</td>
<td>TAIR10</td>
<td>119.15</td>
<td>1533</td>
</tr>
<tr class="even">
<td><a href="https://github.com/tzhu-bio/cisDynet_snakemake/blob/main/blacklist/oryza_sativa_Greenscreen_Freq_100_blacklist.bed">Oryza sativa</a></td>
<td>39947</td>
<td>Japonica rice</td>
<td>IRGSP-1.0</td>
<td>374.47</td>
<td>140</td>
</tr>
<tr class="odd">
<td><a href="https://github.com/tzhu-bio/cisDynet_snakemake/blob/main/blacklist/zea_mays_Greenscreen_Freq_80_blacklist.bed">Zea mays</a></td>
<td>4577</td>
<td>Maize</td>
<td>AGPv3</td>
<td>2066.43</td>
<td>154</td>
</tr>
<tr class="even">
<td><a href="https://github.com/tzhu-bio/cisDynet_snakemake/blob/main/blacklist/solanum_lycopersicum_Greenscreen_Freq_20_blacklist.bed">Solanum lycopersicum</a></td>
<td>4081</td>
<td>Tomato</td>
<td>ITAG2.4</td>
<td>823.94</td>
<td>28</td>
</tr>
<tr class="odd">
<td><a href="https://github.com/tzhu-bio/cisDynet_snakemake/blob/main/blacklist/glycine_max_Greenscreen_Freq_35_blacklist.bed">Glycine max</a></td>
<td>3847</td>
<td>Soybean</td>
<td>Wm82.a2.v1</td>
<td>949.18</td>
<td>53</td>
</tr>
</tbody>
</table>
</div>
<div id="pipeline-output-folders" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Pipeline Output Folders<a href="snakemake-pipeline.html#pipeline-output-folders" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>Important Folders:</strong></p>
<p><em>trimmed</em>: After removing sequencing adapters and low quality sequences.</p>
<p><em>signal</em>: The bigwig file normalized by CPM and RPKM methods.</p>
<p><em>q30</em>: The bam file after removing the low mapping scores and organelle reads.</p>
<p><em>peaks</em>: The narrowpeak files obtained by MACS2.</p>
<p><em>multiqc</em>: The CAT pipeline report.</p>
<p><em>cut_sites</em>: The Tn5 cuts files after shifting the reads.</p>
<p><em>ATACCorrect</em>: Bias correction of ATAC-seq reads in open chromatin using <a href="https://www.nature.com/articles/s41467-020-18035-1">TOBIAS</a></p>
<p><em>BINDetect</em>: Estimation of differentially bound motifs based on scores, sequence and motifs using TOBIAS.</p>
</div>
<div id="report-explanation" class="section level2 hasAnchor" number="3.5">
<h2><span class="header-section-number">3.5</span> Report Explanation<a href="snakemake-pipeline.html#report-explanation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>After running snakemake, we can get the appropriate results as well as an html report. Here is a example <a href="./multiqc_report.html">report</a>.</p>
<p><strong>Duplication rate</strong>: The PCR duplication rate.</p>
<p><strong>FRiP</strong>: Fraction of all mapped reads that fall into the called peak regions, i.e. usable reads in significantly enriched peaks divided by all usable reads.</p>
<p><strong>Mapping rate</strong>: The percentage of reads mapping to genome.</p>
<p><strong>Organell mapping rate</strong>: The percentage of reads mapping to Organelle sequence.</p>
<p><strong>Q30 reads number</strong>: The number of reads after removing low-quality mapping(Default 30).</p>
<p><strong>Raw reads number</strong>: The number of raw reads.</p>
<p><strong>TSS enrichment</strong>: Tn5 cuts enrichment scores around the TSS. See <a href="https://www.encodeproject.org/data-standards/terms/#enrichment">ENCODE</a> for details.</p>
<p><img src="fragments.png" width="100%" />
<em>Left</em>: Typical fragment size distribution plot shows enrichment around 100 and 200 bp, indicating nucleosome-free and mono-nucleosome-bound fragments.</p>
<p><em>Right</em>: Typical TSS enrichment plot shows that nucleosome-free fragments are enriched at TSS.</p>
</div>
<div id="troubleshooting" class="section level2 hasAnchor" number="3.6">
<h2><span class="header-section-number">3.6</span> Troubleshooting<a href="snakemake-pipeline.html#troubleshooting" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ol style="list-style-type: decimal">
<li><strong>Low mapping rate</strong> (usually &lt; 80%): It is possible that there was microbial contamination during the experiment.</li>
<li><strong>High organelle mapping rate</strong> (usually &gt; 20%): A pure suspension of nuclei was not obtained and more washes are recommended. Aspirate all supernatant (which contains mitochondria) as much as possible before performing the transposition reaction. For plant cells, it is highly recommended to use flow cytometry to sort nuclei to reduce contaminants such as organelles.</li>
<li><strong>High duplication rate</strong> (usually &gt; 30%): The library complexity is not enough. The number of cells may not be sufficient. The concentration of DNA at the start of PCR is too low or the number of amplification cycles of PCR is too high. It is recommended to optimise the Tn5 reaction system or reduce the number of cycles.</li>
<li><strong>Low TSS enrichment scores</strong> (usually &lt; 5): The sample may have problems, such as containing dead cells. The suspension contains floating DNA fragments. Prior to the Tn5 transposition reaction, the nuclei were severely broken.</li>
<li><strong>Low FRiP</strong> (usually &lt; 15%): If the experiment is fine, check that the sequencing depth is sufficient.</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="getting-started-with-cisdynet.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="quality-control-plot.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/01-Snakemake_Pipeline.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
