<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 9 Time Series Data | CAT: A toolkit for downstream analysis of bulk ATAC-seq data</title>
  <meta name="description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 9 Time Series Data | CAT: A toolkit for downstream analysis of bulk ATAC-seq data" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 9 Time Series Data | CAT: A toolkit for downstream analysis of bulk ATAC-seq data" />
  
  <meta name="twitter:description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-08-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="tissuesample-specific-analysis.html"/>
<link rel="next" href="motif-enrichment.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Chromatin Accessibility Toolkit</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Primer</a></li>
<li class="chapter" data-level="2" data-path="getting-started-with-cat.html"><a href="getting-started-with-cat.html"><i class="fa fa-check"></i><b>2</b> Getting Started with CAT</a>
<ul>
<li class="chapter" data-level="2.1" data-path="getting-started-with-cat.html"><a href="getting-started-with-cat.html#why-use-cat"><i class="fa fa-check"></i><b>2.1</b> Why use CAT?</a></li>
<li class="chapter" data-level="2.2" data-path="getting-started-with-cat.html"><a href="getting-started-with-cat.html#getting-set-up"><i class="fa fa-check"></i><b>2.2</b> Getting Set Up</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>3</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="3.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#example-data"><i class="fa fa-check"></i><b>3.1</b> Example Data</a></li>
<li class="chapter" data-level="3.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#config-file"><i class="fa fa-check"></i><b>3.2</b> Config File</a></li>
<li class="chapter" data-level="3.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#pipeline-output-folders"><i class="fa fa-check"></i><b>3.3</b> Pipeline Output Folders</a></li>
<li class="chapter" data-level="3.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explanation"><i class="fa fa-check"></i><b>3.4</b> Report Explanation</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="quality-control-plot.html"><a href="quality-control-plot.html"><i class="fa fa-check"></i><b>4</b> Quality Control Plot</a>
<ul>
<li class="chapter" data-level="4.1" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plotfragments"><i class="fa fa-check"></i><b>4.1</b> plotFragments</a></li>
<li class="chapter" data-level="4.2" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plottss"><i class="fa fa-check"></i><b>4.2</b> plotTSS</a></li>
<li class="chapter" data-level="4.3" data-path="quality-control-plot.html"><a href="quality-control-plot.html#quantification"><i class="fa fa-check"></i><b>4.3</b> Quantification</a></li>
<li class="chapter" data-level="4.4" data-path="quality-control-plot.html"><a href="quality-control-plot.html#pca"><i class="fa fa-check"></i><b>4.4</b> PCA</a></li>
<li class="chapter" data-level="4.5" data-path="quality-control-plot.html"><a href="quality-control-plot.html#dendrogram"><i class="fa fa-check"></i><b>4.5</b> Dendrogram</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>5</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotvenn2---6-samples"><i class="fa fa-check"></i><b>5.1</b> PlotVenn(2 - 6 samples)</a></li>
<li class="chapter" data-level="5.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotupset"><i class="fa fa-check"></i><b>5.2</b> plotUpSet</a></li>
<li class="chapter" data-level="5.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>5.3</b> Pairwise module examples</a></li>
<li class="chapter" data-level="5.4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#flower-plot"><i class="fa fa-check"></i><b>5.4</b> Flower plot</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>6</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>6.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="6.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#the-distance-of-peak-summit-to-tss"><i class="fa fa-check"></i><b>6.2</b> The distance of Peak summit to TSS</a></li>
<li class="chapter" data-level="6.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#proximal-distal-intragenic-annotaion"><i class="fa fa-check"></i><b>6.3</b> Proximal / Distal / Intragenic Annotaion</a></li>
<li class="chapter" data-level="6.4" data-path="peak-annotaion.html"><a href="peak-annotaion.html#annotation-for-merged-peaks"><i class="fa fa-check"></i><b>6.4</b> Annotation for merged peaks</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html"><i class="fa fa-check"></i><b>7</b> Differential OCR analysis</a>
<ul>
<li class="chapter" data-level="7.1" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups"><i class="fa fa-check"></i><b>7.1</b> Using DEseq2 to identify differentail OCR (Two groups)</a></li>
<li class="chapter" data-level="7.2" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes"><i class="fa fa-check"></i><b>7.2</b> Get the differential peaks target genes</a></li>
<li class="chapter" data-level="7.3" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#plotma"><i class="fa fa-check"></i><b>7.3</b> PlotMA</a></li>
<li class="chapter" data-level="7.4" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples"><i class="fa fa-check"></i><b>7.4</b> Compare the three samples chromatin accessibility (Three samples)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html"><i class="fa fa-check"></i><b>8</b> Tissue(Sample) Specific Analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix"><i class="fa fa-check"></i><b>8.1</b> Calaluate the SPM matrix</a></li>
<li class="chapter" data-level="8.2" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests"><i class="fa fa-check"></i><b>8.2</b> Select the specific peak with your interests</a></li>
<li class="chapter" data-level="8.3" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score"><i class="fa fa-check"></i><b>8.3</b> Select the specifc peaks with JSD score</a></li>
<li class="chapter" data-level="8.4" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues"><i class="fa fa-check"></i><b>8.4</b> Get the Top N specific peaks in every Samples / Tissues</a></li>
<li class="chapter" data-level="8.5" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number"><i class="fa fa-check"></i><b>8.5</b> Get the optimal cluster number</a></li>
<li class="chapter" data-level="8.6" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes"><i class="fa fa-check"></i><b>8.6</b> Get the cluster target genes</a></li>
<li class="chapter" data-level="8.7" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#go-enrichment"><i class="fa fa-check"></i><b>8.7</b> GO Enrichment</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="time-series-data.html"><a href="time-series-data.html"><i class="fa fa-check"></i><b>9</b> Time Series Data</a>
<ul>
<li class="chapter" data-level="9.1" data-path="time-series-data.html"><a href="time-series-data.html#fitting-time-with-atac-seq"><i class="fa fa-check"></i><b>9.1</b> Fitting time with ATAC-seq</a></li>
<li class="chapter" data-level="9.2" data-path="time-series-data.html"><a href="time-series-data.html#integration-with-rna-seq"><i class="fa fa-check"></i><b>9.2</b> Integration with RNA-seq</a></li>
<li class="chapter" data-level="9.3" data-path="time-series-data.html"><a href="time-series-data.html#get-fitting-time-genes"><i class="fa fa-check"></i><b>9.3</b> Get fitting time genes</a></li>
<li class="chapter" data-level="9.4" data-path="time-series-data.html"><a href="time-series-data.html#integration-with-atac-rna-seq"><i class="fa fa-check"></i><b>9.4</b> Integration with ATAC / RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>10</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="10.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#chromvar"><i class="fa fa-check"></i><b>10.1</b> ChromVAR</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="footprint-analysis.html"><a href="footprint-analysis.html"><i class="fa fa-check"></i><b>11</b> Footprint Analysis</a>
<ul>
<li class="chapter" data-level="11.1" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-footprint-score"><i class="fa fa-check"></i><b>11.1</b> Get footprint score</a></li>
<li class="chapter" data-level="11.2" data-path="footprint-analysis.html"><a href="footprint-analysis.html#plot-footprint-score"><i class="fa fa-check"></i><b>11.2</b> Plot footprint score</a></li>
<li class="chapter" data-level="11.3" data-path="footprint-analysis.html"><a href="footprint-analysis.html#compare-footprints"><i class="fa fa-check"></i><b>11.3</b> Compare Footprints</a></li>
<li class="chapter" data-level="11.4" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-differential-footprints"><i class="fa fa-check"></i><b>11.4</b> Get Differential Footprints</a></li>
<li class="chapter" data-level="11.5" data-path="footprint-analysis.html"><a href="footprint-analysis.html#getmotif2gene"><i class="fa fa-check"></i><b>11.5</b> getMotif2Gene</a></li>
<li class="chapter" data-level="11.6" data-path="footprint-analysis.html"><a href="footprint-analysis.html#integration-with-rna-seq-1"><i class="fa fa-check"></i><b>11.6</b> Integration with RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="co-accessibility.html"><a href="co-accessibility.html"><i class="fa fa-check"></i><b>12</b> Co-accessibility</a>
<ul>
<li class="chapter" data-level="12.1" data-path="co-accessibility.html"><a href="co-accessibility.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>12.1</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="12.2" data-path="co-accessibility.html"><a href="co-accessibility.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>12.2</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>13</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="13.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#get-the-peak-target-genes"><i class="fa fa-check"></i><b>13.1</b> Get the peak target genes</a></li>
<li class="chapter" data-level="13.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plot-all-the-significant-peak2gene-links"><i class="fa fa-check"></i><b>13.2</b> Plot all the significant peak2gene links</a></li>
<li class="chapter" data-level="13.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>13.3</b> Link OCR to genes</a></li>
<li class="chapter" data-level="13.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotp2gtracks"><i class="fa fa-check"></i><b>13.4</b> plotP2GTracks</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>14</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="14.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#motif2gene"><i class="fa fa-check"></i><b>14.1</b> Motif2Gene</a></li>
<li class="chapter" data-level="14.2" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#building-regulatory-network-1"><i class="fa fa-check"></i><b>14.2</b> Building regulatory network</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html"><i class="fa fa-check"></i><b>15</b> GWAS Trait SNPs Enrichment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#workflows"><i class="fa fa-check"></i><b>15.1</b> Workflows</a></li>
<li class="chapter" data-level="15.2" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#preprocessing"><i class="fa fa-check"></i><b>15.2</b> Preprocessing</a></li>
<li class="chapter" data-level="15.3" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#do-enrichment"><i class="fa fa-check"></i><b>15.3</b> Do Enrichment</a></li>
<li class="chapter" data-level="15.4" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#plotting"><i class="fa fa-check"></i><b>15.4</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="other-utils.html"><a href="other-utils.html"><i class="fa fa-check"></i><b>16</b> Other Utils</a>
<ul>
<li class="chapter" data-level="16.1" data-path="other-utils.html"><a href="other-utils.html#super-enhancer"><i class="fa fa-check"></i><b>16.1</b> Super Enhancer</a></li>
<li class="chapter" data-level="16.2" data-path="other-utils.html"><a href="other-utils.html#peak-fisher-test"><i class="fa fa-check"></i><b>16.2</b> Peak Fisher Test</a></li>
<li class="chapter" data-level="16.3" data-path="other-utils.html"><a href="other-utils.html#snp2gene"><i class="fa fa-check"></i><b>16.3</b> SNP2Gene</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="session-information.html"><a href="session-information.html"><i class="fa fa-check"></i><b>17</b> Session Information</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">CAT: A toolkit for downstream analysis of bulk ATAC-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="time-series-data" class="section level1 hasAnchor" number="9">
<h1><span class="header-section-number">Chapter 9</span> Time Series Data<a href="time-series-data.html#time-series-data" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Biological time series data are a common experimental design in research and are important for understanding the dynamic processes of biological systems, revealing regulatory mechanisms, predicting trends, and individual differences and responses.</p>
<p><strong>Time-series ATAC-seq</strong> can help us decipher the following aspects:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Dynamic chromatin accessibility:</strong> Time-series ATAC-seq enables the investigation of dynamic changes in chromatin accessibility over time. It allows us to identify genomic regions that undergo opening or closing during specific time points, shedding light on the temporal regulation of chromatin structure.</p></li>
<li><p><strong>Transcriptional regulation:</strong> By comparing ATAC-seq profiles across multiple time points, we can unravel the regulatory elements and transcription factor binding sites associated with specific temporal patterns of chromatin accessibility. This information provides insights into the dynamic regulation of gene expression during different stages or conditions.</p></li>
<li><p><strong>Cell fate determination:</strong> Time-series ATAC-seq data facilitates the exploration of chromatin dynamics during cellular differentiation and development. It helps uncover the changes in chromatin accessibility associated with cell fate decisions and provides a molecular understanding of the underlying mechanisms governing cell fate determination.</p></li>
<li><p><strong>Temporal gene regulatory networks:</strong> Integrating time-series ATAC-seq data with gene expression data allows the construction of temporal gene regulatory networks. These networks provide a comprehensive view of the regulatory interactions and signaling pathways involved in orchestrating gene expression changes over time.</p></li>
<li><p><strong>Disease progression and response:</strong> Time-series ATAC-seq analysis can elucidate the dynamic alterations in chromatin accessibility associated with disease progression, treatment response, or cellular response to external stimuli. It aids in identifying key regulatory elements and pathways implicated in disease development and therapeutic interventions.</p></li>
</ol>
<p>In summary, time-series ATAC-seq data provides a powerful tool to decipher the dynamic nature of chromatin accessibility and its impact on gene regulation, cell fate determination, and disease processes. By capturing temporal changes in chromatin accessibility, we gain valuable insights into the underlying molecular mechanisms driving biological processes and disease progression.</p>
<p>When analyzing changes over time in ATAC-seq data, our primary focus is on identifying the OCRs that undergo changes throughout the process. Therefore, our input should consist of the differential OCRs between each pair of time points. Unlike the pseudotime for single cells, our bulk ATAC-seq data is generated at predetermined time points, eliminating the need to infer the temporal order between samples.</p>
<div id="fitting-time-with-atac-seq" class="section level2 hasAnchor" number="9.1">
<h2><span class="header-section-number">9.1</span> Fitting time with ATAC-seq<a href="time-series-data.html#fitting-time-with-atac-seq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Thus for normalized quantitative data, we first use <strong>getSpecificPeak</strong> or <strong>getTopSpecifcPeaks</strong> to get dynamic difference OCRs.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="time-series-data.html#cb1-1" aria-hidden="true" tabindex="-1"></a>dynamic_ocrs <span class="ot">&lt;-</span> <span class="fu">getTopSpecifcPeaks</span>(<span class="at">spm_data =</span> spm, <span class="at">norm_data =</span> quant_data, <span class="at">top_N =</span> <span class="dv">1000</span>, <span class="at">save_path =</span> <span class="st">&quot;F:/CAT/example/&quot;</span>, <span class="at">file_prefix =</span> <span class="st">&quot;Top1000_raw_data&quot;</span>)</span></code></pre></div>
<p>We then use <strong>getTimeATAC</strong> to fit these dynamic OCRs in order of time points.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="time-series-data.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">getTimeATAC</span>(<span class="at">norm_data =</span> dynamic_ocrs)</span></code></pre></div>
<pre><code>## `use_raster` is automatically set to TRUE for a matrix with more than
## 2000 rows. You can control `use_raster` argument by explicitly setting
## TRUE/FALSE to it.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<pre><code>## &#39;magick&#39; package is suggested to install to give better rasterization.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<p><img src="bookdown-demo_files/figure-html/getTimeATAC-1.png" width="288" style="display: block; margin: auto;" /></p>
</div>
<div id="integration-with-rna-seq" class="section level2 hasAnchor" number="9.2">
<h2><span class="header-section-number">9.2</span> Integration with RNA-seq<a href="time-series-data.html#integration-with-rna-seq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Of course we can also use the information from peak2gene to get the current dynamic OCR corresponding to the expression change of the target gene.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="time-series-data.html#cb5-1" aria-hidden="true" tabindex="-1"></a>peak_time <span class="ot">&lt;-</span> <span class="fu">getTimeATAC</span>(<span class="at">norm_data =</span> dynamic_ocrs, <span class="at">return_matrix =</span> T)</span>
<span id="cb5-2"><a href="time-series-data.html#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="fu">getTimeRNA</span>(<span class="at">peak_time =</span> peak_time, <span class="at">peak2gene =</span> <span class="st">&quot;F:/CAT/example/All_Peak2Gene_links.rds&quot;</span>,</span>
<span id="cb5-3"><a href="time-series-data.html#cb5-3" aria-hidden="true" tabindex="-1"></a>           <span class="at">rna_matrix =</span> <span class="st">&quot;F:/CAT/example/RNA_TPM_Norm_Data.tsv&quot;</span>,<span class="at">corr_cutoff =</span> <span class="fl">0.4</span>)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/getTimeRNA-1.png" width="288" style="display: block; margin: auto;" /></p>
</div>
<div id="get-fitting-time-genes" class="section level2 hasAnchor" number="9.3">
<h2><span class="header-section-number">9.3</span> Get fitting time genes<a href="time-series-data.html#get-fitting-time-genes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We can use <strong>getTimeGene</strong> to extract a list of dynamically changing genes.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="time-series-data.html#cb6-1" aria-hidden="true" tabindex="-1"></a>gene <span class="ot">&lt;-</span> <span class="fu">getTimeGene</span>(<span class="at">peak_time =</span> peak_time, <span class="at">peak2gene =</span> <span class="st">&quot;F:/CAT/example/All_Peak2Gene_links.rds&quot;</span>,</span>
<span id="cb6-2"><a href="time-series-data.html#cb6-2" aria-hidden="true" tabindex="-1"></a>                    <span class="at">rna_matrix =</span> <span class="st">&quot;F:/CAT/example/RNA_TPM_Norm_Data.tsv&quot;</span>,<span class="at">corr_cutoff =</span> <span class="fl">0.4</span>)</span>
<span id="cb6-3"><a href="time-series-data.html#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(gene)</span></code></pre></div>
<pre><code>##              Gene                      Peak
## 1 ENSG00000149476   chr11:61124602-61125215
## 2 ENSG00000257758 chr12:105368383-105368882
## 3 ENSG00000232557    chr1:24239719-24240067
## 4 ENSG00000012779   chr10:45871188-45871634
## 5 ENSG00000137673 chr11:102417168-102417481
## 6 ENSG00000142583      chr1:9129424-9130084</code></pre>
</div>
<div id="integration-with-atac-rna-seq" class="section level2 hasAnchor" number="9.4">
<h2><span class="header-section-number">9.4</span> Integration with ATAC / RNA-seq<a href="time-series-data.html#integration-with-atac-rna-seq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We label the right side of the heatmap based on the genes obtained above. You need to prepare a file in this format.</p>
<pre><code>##              gene    name
## 1 ENSG00000085831  TTC39A
## 2 ENSG00000150316   CWC15
## 3 ENSG00000169504   CLIC4
## 4 ENSG00000161955 TNFSF13
## 5 ENSG00000095585    BLNK
## 6 ENSG00000169607  CKAP2L</code></pre>
<p>Next we can use the <strong>plotTimeAll</strong> function to draw the heatmap of ATAC and RNA at the same time and label some genes.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="time-series-data.html#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotTimeAll</span>(<span class="at">norm_data =</span> dynamic_ocrs, </span>
<span id="cb9-2"><a href="time-series-data.html#cb9-2" aria-hidden="true" tabindex="-1"></a>            <span class="at">peak2gene =</span> <span class="st">&quot;F:/CAT/example/All_Peak2Gene_links.rds&quot;</span>,</span>
<span id="cb9-3"><a href="time-series-data.html#cb9-3" aria-hidden="true" tabindex="-1"></a>            <span class="at">rna_matrix =</span> <span class="st">&quot;F:/CAT/example/RNA_TPM_Norm_Data.tsv&quot;</span>,</span>
<span id="cb9-4"><a href="time-series-data.html#cb9-4" aria-hidden="true" tabindex="-1"></a>            <span class="at">label =</span> <span class="st">&quot;F:/CAT/example/time_anno.txt&quot;</span>)</span></code></pre></div>
<pre><code>## 2023-08-21 10:53:40 Calculating ATAC time...</code></pre>
<pre><code>## 2023-08-21 10:53:55 Calculating RNA time...</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotTimeAll-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>We can also use <strong>plotTimeGene</strong> to draw dynamic changes in the expression of individual genes.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="time-series-data.html#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotTimeGene</span>(<span class="at">peak_time =</span> peak_time,<span class="at">peak2gene =</span> <span class="st">&quot;F:/CAT/example/All_Peak2Gene_links.rds&quot;</span>,</span>
<span id="cb12-2"><a href="time-series-data.html#cb12-2" aria-hidden="true" tabindex="-1"></a>             <span class="at">rna_matrix =</span> <span class="st">&quot;F:/CAT/example/RNA_TPM_Norm_Data.tsv&quot;</span>,<span class="at">corr_cutoff =</span> <span class="fl">0.4</span>,<span class="at">gene =</span> <span class="st">&quot;ENSG00000234425&quot;</span>)</span></code></pre></div>
<pre><code>## `geom_smooth()` using method = &#39;gam&#39; and formula = &#39;y ~ s(x, bs = &quot;cs&quot;)&#39;</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotTimeGene-1.png" width="672" /></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="tissuesample-specific-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="motif-enrichment.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/07-TimeSeries.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
