<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 8 Tissue(Sample) Specific Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</title>
  <meta name="description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 8 Tissue(Sample) Specific Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="cis-regulatory chromatin dynamics and networks." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 8 Tissue(Sample) Specific Analysis | cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks" />
  
  <meta name="twitter:description" content="cis-regulatory chromatin dynamics and networks." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-08-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="differential-ocr-analysis.html"/>
<link rel="next" href="time-course-data.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">cisDynet</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Primer</a></li>
<li class="chapter" data-level="2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html"><i class="fa fa-check"></i><b>2</b> Getting Started with cisDynet</a>
<ul>
<li class="chapter" data-level="2.1" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#why-use-cisdynet"><i class="fa fa-check"></i><b>2.1</b> Why use cisDynet?</a></li>
<li class="chapter" data-level="2.2" data-path="getting-started-with-cisdynet.html"><a href="getting-started-with-cisdynet.html#getting-set-up"><i class="fa fa-check"></i><b>2.2</b> Getting Set Up</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>3</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="3.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#example-data"><i class="fa fa-check"></i><b>3.1</b> Example Data</a></li>
<li class="chapter" data-level="3.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#config-file"><i class="fa fa-check"></i><b>3.2</b> Config File</a></li>
<li class="chapter" data-level="3.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#blacklists"><i class="fa fa-check"></i><b>3.3</b> Blacklists</a></li>
<li class="chapter" data-level="3.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#pipeline-output-folders"><i class="fa fa-check"></i><b>3.4</b> Pipeline Output Folders</a></li>
<li class="chapter" data-level="3.5" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explanation"><i class="fa fa-check"></i><b>3.5</b> Report Explanation</a></li>
<li class="chapter" data-level="3.6" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#troubleshooting"><i class="fa fa-check"></i><b>3.6</b> Troubleshooting</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="quality-control-plot.html"><a href="quality-control-plot.html"><i class="fa fa-check"></i><b>4</b> Quality Control Plot</a>
<ul>
<li class="chapter" data-level="4.1" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plotfragments"><i class="fa fa-check"></i><b>4.1</b> plotFragments</a></li>
<li class="chapter" data-level="4.2" data-path="quality-control-plot.html"><a href="quality-control-plot.html#plottss"><i class="fa fa-check"></i><b>4.2</b> plotTSS</a></li>
<li class="chapter" data-level="4.3" data-path="quality-control-plot.html"><a href="quality-control-plot.html#quantification"><i class="fa fa-check"></i><b>4.3</b> Quantification</a></li>
<li class="chapter" data-level="4.4" data-path="quality-control-plot.html"><a href="quality-control-plot.html#pca"><i class="fa fa-check"></i><b>4.4</b> PCA</a></li>
<li class="chapter" data-level="4.5" data-path="quality-control-plot.html"><a href="quality-control-plot.html#dendrogram"><i class="fa fa-check"></i><b>4.5</b> Dendrogram</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>5</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotvenn2---6-samples"><i class="fa fa-check"></i><b>5.1</b> PlotVenn(2 - 6 samples)</a></li>
<li class="chapter" data-level="5.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#plotupset"><i class="fa fa-check"></i><b>5.2</b> plotUpSet</a></li>
<li class="chapter" data-level="5.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>5.3</b> Pairwise module examples</a></li>
<li class="chapter" data-level="5.4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#flower-plot"><i class="fa fa-check"></i><b>5.4</b> Flower plot</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>6</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>6.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="6.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#the-distance-of-peak-summit-to-tss"><i class="fa fa-check"></i><b>6.2</b> The distance of Peak summit to TSS</a></li>
<li class="chapter" data-level="6.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#proximal-distal-intragenic-annotaion"><i class="fa fa-check"></i><b>6.3</b> Proximal / Distal / Intragenic Annotaion</a></li>
<li class="chapter" data-level="6.4" data-path="peak-annotaion.html"><a href="peak-annotaion.html#annotation-for-merged-peaks"><i class="fa fa-check"></i><b>6.4</b> Annotation for merged peaks</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html"><i class="fa fa-check"></i><b>7</b> Differential OCR analysis</a>
<ul>
<li class="chapter" data-level="7.1" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#using-deseq2-to-identify-differentail-ocr-two-groups"><i class="fa fa-check"></i><b>7.1</b> Using DEseq2 to identify differentail OCR (Two groups)</a></li>
<li class="chapter" data-level="7.2" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#get-the-differential-peaks-target-genes"><i class="fa fa-check"></i><b>7.2</b> Get the differential peaks target genes</a></li>
<li class="chapter" data-level="7.3" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#plotma"><i class="fa fa-check"></i><b>7.3</b> PlotMA</a></li>
<li class="chapter" data-level="7.4" data-path="differential-ocr-analysis.html"><a href="differential-ocr-analysis.html#compare-the-three-samples-chromatin-accessibility-three-samples"><i class="fa fa-check"></i><b>7.4</b> Compare the three samples chromatin accessibility (Three samples)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html"><i class="fa fa-check"></i><b>8</b> Tissue(Sample) Specific Analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix"><i class="fa fa-check"></i><b>8.1</b> Calaluate the SPM matrix</a></li>
<li class="chapter" data-level="8.2" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests"><i class="fa fa-check"></i><b>8.2</b> Select the specific peak with your interests</a></li>
<li class="chapter" data-level="8.3" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score"><i class="fa fa-check"></i><b>8.3</b> Select the specifc peaks with JSD score</a></li>
<li class="chapter" data-level="8.4" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues"><i class="fa fa-check"></i><b>8.4</b> Get the Top N specific peaks in every Samples / Tissues</a></li>
<li class="chapter" data-level="8.5" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number"><i class="fa fa-check"></i><b>8.5</b> Get the optimal cluster number</a></li>
<li class="chapter" data-level="8.6" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes"><i class="fa fa-check"></i><b>8.6</b> Get the cluster target genes</a></li>
<li class="chapter" data-level="8.7" data-path="tissuesample-specific-analysis.html"><a href="tissuesample-specific-analysis.html#go-enrichment"><i class="fa fa-check"></i><b>8.7</b> GO Enrichment</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="time-course-data.html"><a href="time-course-data.html"><i class="fa fa-check"></i><b>9</b> Time Course Data</a>
<ul>
<li class="chapter" data-level="9.1" data-path="time-course-data.html"><a href="time-course-data.html#fitting-time-with-atac-seq"><i class="fa fa-check"></i><b>9.1</b> Fitting time with ATAC-seq</a></li>
<li class="chapter" data-level="9.2" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-rna-seq"><i class="fa fa-check"></i><b>9.2</b> Integration with RNA-seq</a></li>
<li class="chapter" data-level="9.3" data-path="time-course-data.html"><a href="time-course-data.html#get-fitting-time-genes"><i class="fa fa-check"></i><b>9.3</b> Get fitting time genes</a></li>
<li class="chapter" data-level="9.4" data-path="time-course-data.html"><a href="time-course-data.html#integration-with-atac-rna-seq"><i class="fa fa-check"></i><b>9.4</b> Integration with ATAC / RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>10</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="10.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#chromvar"><i class="fa fa-check"></i><b>10.1</b> ChromVAR</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="footprint-analysis.html"><a href="footprint-analysis.html"><i class="fa fa-check"></i><b>11</b> Footprint Analysis</a>
<ul>
<li class="chapter" data-level="11.1" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-footprint-score"><i class="fa fa-check"></i><b>11.1</b> Get footprint score</a></li>
<li class="chapter" data-level="11.2" data-path="footprint-analysis.html"><a href="footprint-analysis.html#plot-footprint-score"><i class="fa fa-check"></i><b>11.2</b> Plot footprint score</a></li>
<li class="chapter" data-level="11.3" data-path="footprint-analysis.html"><a href="footprint-analysis.html#compare-footprints"><i class="fa fa-check"></i><b>11.3</b> Compare Footprints</a></li>
<li class="chapter" data-level="11.4" data-path="footprint-analysis.html"><a href="footprint-analysis.html#get-differential-footprints"><i class="fa fa-check"></i><b>11.4</b> Get Differential Footprints</a></li>
<li class="chapter" data-level="11.5" data-path="footprint-analysis.html"><a href="footprint-analysis.html#getmotif2gene"><i class="fa fa-check"></i><b>11.5</b> getMotif2Gene</a></li>
<li class="chapter" data-level="11.6" data-path="footprint-analysis.html"><a href="footprint-analysis.html#integration-with-rna-seq-1"><i class="fa fa-check"></i><b>11.6</b> Integration with RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="co-accessibility.html"><a href="co-accessibility.html"><i class="fa fa-check"></i><b>12</b> Co-accessibility</a>
<ul>
<li class="chapter" data-level="12.1" data-path="co-accessibility.html"><a href="co-accessibility.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>12.1</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="12.2" data-path="co-accessibility.html"><a href="co-accessibility.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>12.2</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>13</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="13.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#get-the-peak-target-genes"><i class="fa fa-check"></i><b>13.1</b> Get the peak target genes</a></li>
<li class="chapter" data-level="13.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plot-all-the-significant-peak2gene-links"><i class="fa fa-check"></i><b>13.2</b> Plot all the significant peak2gene links</a></li>
<li class="chapter" data-level="13.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>13.3</b> Link OCR to genes</a></li>
<li class="chapter" data-level="13.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotp2gtracks"><i class="fa fa-check"></i><b>13.4</b> plotP2GTracks</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>14</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="14.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#motif2gene"><i class="fa fa-check"></i><b>14.1</b> Motif2Gene</a></li>
<li class="chapter" data-level="14.2" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#building-regulatory-network-1"><i class="fa fa-check"></i><b>14.2</b> Building regulatory network</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html"><i class="fa fa-check"></i><b>15</b> GWAS Trait SNPs Enrichment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#workflows"><i class="fa fa-check"></i><b>15.1</b> Workflows</a></li>
<li class="chapter" data-level="15.2" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#preprocessing"><i class="fa fa-check"></i><b>15.2</b> Preprocessing</a></li>
<li class="chapter" data-level="15.3" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#do-enrichment"><i class="fa fa-check"></i><b>15.3</b> Do Enrichment</a></li>
<li class="chapter" data-level="15.4" data-path="gwas-trait-snps-enrichment.html"><a href="gwas-trait-snps-enrichment.html#plotting"><i class="fa fa-check"></i><b>15.4</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="other-utils.html"><a href="other-utils.html"><i class="fa fa-check"></i><b>16</b> Other Utils</a>
<ul>
<li class="chapter" data-level="16.1" data-path="other-utils.html"><a href="other-utils.html#super-enhancer"><i class="fa fa-check"></i><b>16.1</b> Super Enhancer</a></li>
<li class="chapter" data-level="16.2" data-path="other-utils.html"><a href="other-utils.html#peak-fisher-test"><i class="fa fa-check"></i><b>16.2</b> Peak Fisher Test</a></li>
<li class="chapter" data-level="16.3" data-path="other-utils.html"><a href="other-utils.html#snp2gene"><i class="fa fa-check"></i><b>16.3</b> SNP2Gene</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="session-information.html"><a href="session-information.html"><i class="fa fa-check"></i><b>17</b> Session Information</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">cisDynet：an integrated platform for modeling gene-regulatory dynamics and networks</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="tissuesample-specific-analysis" class="section level1 hasAnchor" number="8">
<h1><span class="header-section-number">Chapter 8</span> Tissue(Sample) Specific Analysis<a href="tissuesample-specific-analysis.html#tissuesample-specific-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Tissue/organ-specific ATAC-seq OCRs can help us identify and understand differences between cell types and tissues. By analyzing specific OCRs, we can identify genes and regulatory elements that are highly expressed in a particular tissue/organ, providing insight into the basis of cell type and tissue specificity.</p>
<div id="calaluate-the-spm-matrix" class="section level2 hasAnchor" number="8.1">
<h2><span class="header-section-number">8.1</span> Calaluate the SPM matrix<a href="tissuesample-specific-analysis.html#calaluate-the-spm-matrix" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>For the quantified and normalized matrix, we can use <strong>calSPM</strong> to simultaneously calculate the specificity score of each OCR within each sample. Higher scores represent higher specificity for that OCR.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="tissuesample-specific-analysis.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;All_quant_data&quot;</span>)</span>
<span id="cb1-2"><a href="tissuesample-specific-analysis.html#cb1-2" aria-hidden="true" tabindex="-1"></a>quant_data <span class="ot">&lt;-</span> All_quant_data</span>
<span id="cb1-3"><a href="tissuesample-specific-analysis.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">calSPM</span>(quant_data, <span class="at">plot =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>## No id variables; using all as measure variables</code></pre>
<p><img src="bookdown-demo_files/figure-html/calSPM-1.png" width="672" /></p>
</div>
<div id="select-the-specific-peak-with-your-interests" class="section level2 hasAnchor" number="8.2">
<h2><span class="header-section-number">8.2</span> Select the specific peak with your interests<a href="tissuesample-specific-analysis.html#select-the-specific-peak-with-your-interests" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In this way, we can obtain a new matrix of the same size as the input matrix, except that the values in it represent SPM scores, and by filtering this SPM score matrix, we can arbitrarily obtain OCRs that are either specifically open or both highly open in one or more samples. Here, we filtered out the highly open OCRs in <em>Bulk_B</em>, <em>Mem B</em> and <em>Naive_B</em>.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="tissuesample-specific-analysis.html#cb3-1" aria-hidden="true" tabindex="-1"></a>spm <span class="ot">&lt;-</span> <span class="fu">calSPM</span>(quant_data, <span class="at">plot =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
<pre><code>## No id variables; using all as measure variables</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="tissuesample-specific-analysis.html#cb5-1" aria-hidden="true" tabindex="-1"></a>peak <span class="ot">&lt;-</span> spm[spm<span class="sc">$</span>Bulk_B <span class="sc">&gt;=</span> <span class="fl">0.4</span> <span class="sc">&amp;</span> spm<span class="sc">$</span>Mem_B <span class="sc">&gt;=</span> <span class="fl">0.4</span> <span class="sc">&amp;</span> spm<span class="sc">$</span>Naive_B <span class="sc">&gt;</span> <span class="fl">0.4</span>,]</span>
<span id="cb5-2"><a href="tissuesample-specific-analysis.html#cb5-2" aria-hidden="true" tabindex="-1"></a>test <span class="ot">&lt;-</span> quant_data[<span class="fu">rownames</span>(peak),]</span>
<span id="cb5-3"><a href="tissuesample-specific-analysis.html#cb5-3" aria-hidden="true" tabindex="-1"></a>pheatmap<span class="sc">::</span><span class="fu">pheatmap</span>(<span class="fu">rowZscores</span>(<span class="fu">as.matrix</span>(test)),<span class="at">show_rownames =</span> F)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/Select-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="select-the-specifc-peaks-with-jsd-score" class="section level2 hasAnchor" number="8.3">
<h2><span class="header-section-number">8.3</span> Select the specifc peaks with JSD score<a href="tissuesample-specific-analysis.html#select-the-specifc-peaks-with-jsd-score" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>In addition to this, we can also use <strong>calJSD</strong> screen differential OCRs without bias by calculating the JSD score of each OCR across samples.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="tissuesample-specific-analysis.html#cb6-1" aria-hidden="true" tabindex="-1"></a>jsd_res <span class="ot">&lt;-</span> <span class="fu">calJSD</span>(<span class="at">norm_data=</span>quant_data)</span></code></pre></div>
<pre><code>## 2023-10-23 14:36:28 Calculating JSD...</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="tissuesample-specific-analysis.html#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(jsd_res)</span></code></pre></div>
<pre><code>##                    H_specificity
## chr1:9910-10596     4.358698e-02
## chr1:564574-570175  1.048062e-07
## chr1:713757-714733  8.826571e-03
## chr1:750692-750842  5.125147e-01
## chr1:752562-752915  6.348925e-02
## chr1:762339-763292  2.164577e-02</code></pre>
<p>Draw a density plot to see the distribution of JSD scores. Draw a density plot to see the distribution of JSD score. Similarly, a larger value of JSD score for OCR represents more specificity.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="tissuesample-specific-analysis.html#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb10-2"><a href="tissuesample-specific-analysis.html#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpubr)</span>
<span id="cb10-3"><a href="tissuesample-specific-analysis.html#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(jsd_res, <span class="fu">aes</span>(<span class="at">x=</span>H_specificity)) <span class="sc">+</span> </span>
<span id="cb10-4"><a href="tissuesample-specific-analysis.html#cb10-4" aria-hidden="true" tabindex="-1"></a>        <span class="fu">geom_density</span>(<span class="at">color=</span> <span class="st">&quot;#08519C&quot;</span>, <span class="at">fill=</span><span class="st">&quot;#DEEBF7&quot;</span>, <span class="at">size=</span><span class="dv">1</span>) <span class="sc">+</span> <span class="fu">theme_pubr</span>() <span class="sc">+</span></span>
<span id="cb10-5"><a href="tissuesample-specific-analysis.html#cb10-5" aria-hidden="true" tabindex="-1"></a>        <span class="fu">xlab</span>(<span class="st">&quot;JSD score&quot;</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">&quot;Density&quot;</span>)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plot%20jsd-1.png" width="384" style="display: block; margin: auto;" /></p>
<p>Next we can use <strong>getSpecificPeak</strong> to obtain tissue-specific OCRs.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="tissuesample-specific-analysis.html#cb11-1" aria-hidden="true" tabindex="-1"></a>sp_peaks <span class="ot">&lt;-</span> <span class="fu">getSpecificPeak</span>(<span class="at">norm_data =</span> quant_data, <span class="at">jsd_file =</span> jsd_res, <span class="at">cutoff =</span> <span class="fl">0.2</span>)</span></code></pre></div>
<pre><code>## 2023-10-23 14:36:33 Get the TOP 20.00% specifci peaks.</code></pre>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="tissuesample-specific-analysis.html#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">dim</span>(sp_peaks)</span></code></pre></div>
<pre><code>## [1] 23740    20</code></pre>
<p>We can use <strong>plotSpecificPeak</strong> to draw heatmaps of specific OCR. However, we found that most of these specific OCRs are derived from Plasmacytoid dendritic cells (pDCs), this is because our samples are mainly derived from T-cell types, and the differences between different T-cells are definitely smaller than the differences between different cell types. So it seems more reasonable to use TOP-specific OCRs in each sample for this analysis where the original sample is biased.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="tissuesample-specific-analysis.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotSpecificPeak</span>(<span class="at">specific_peak =</span> sp_peaks)</span></code></pre></div>
<pre><code>## `use_raster` is automatically set to TRUE for a matrix with more than
## 2000 rows. You can control `use_raster` argument by explicitly setting
## TRUE/FALSE to it.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<pre><code>## &#39;magick&#39; package is suggested to install to give better rasterization.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotSpecificPeak-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="get-the-top-n-specific-peaks-in-every-samples-tissues" class="section level2 hasAnchor" number="8.4">
<h2><span class="header-section-number">8.4</span> Get the Top N specific peaks in every Samples / Tissues<a href="tissuesample-specific-analysis.html#get-the-top-n-specific-peaks-in-every-samples-tissues" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>So we provide the <strong>getTopSpecifcPeaks</strong> function to help you select TOP-specific OCRs by samples. It is worth noting that top-specific OCRs for similar samples may also be shared. So the number of TOP-specific OCRs you get at the end may not be equal to (in most cases is less than) <em>N</em><sub>sample</sub> x <em>N</em><sub>top</sub>.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="tissuesample-specific-analysis.html#cb18-1" aria-hidden="true" tabindex="-1"></a>top1000 <span class="ot">&lt;-</span> <span class="fu">getTopSpecifcPeaks</span>(<span class="at">spm_data =</span> spm, <span class="at">norm_data =</span> quant_data, <span class="at">top_N =</span> <span class="dv">1000</span>, <span class="at">save_path =</span> <span class="st">&quot;F:/cisDynet/example/&quot;</span>, <span class="at">file_prefix =</span> <span class="st">&quot;Top2000_raw_data&quot;</span>)</span>
<span id="cb18-2"><a href="tissuesample-specific-analysis.html#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="fu">dim</span>(top1000)</span></code></pre></div>
<pre><code>## [1] 10496    20</code></pre>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="tissuesample-specific-analysis.html#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotSpecificPeak</span>(<span class="at">specific_peak =</span> top1000)</span></code></pre></div>
<pre><code>## `use_raster` is automatically set to TRUE for a matrix with more than
## 2000 rows. You can control `use_raster` argument by explicitly setting
## TRUE/FALSE to it.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<pre><code>## &#39;magick&#39; package is suggested to install to give better rasterization.
## 
## Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<p><img src="bookdown-demo_files/figure-html/plotSpecificPeak2-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="get-the-optimal-cluster-number" class="section level2 hasAnchor" number="8.5">
<h2><span class="header-section-number">8.5</span> Get the optimal cluster number<a href="tissuesample-specific-analysis.html#get-the-optimal-cluster-number" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Next, we can go ahead and divide these obtained difference OCRs into clusters/modules. We start by determining how many clusters/modules are appropriate to split based on <strong>getClusterNum</strong>. We generally use the final number of clusters/modules based on their elbow point. Of course, this step is <em>NOT necessary</em>. You can set a reasonable number of clusters/modules, as long as most of them are biologically meaningful.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="tissuesample-specific-analysis.html#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">getClusterNum</span>(top1000)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/calClusterNum-1.png" width="384" style="display: block; margin: auto;" /></p>
<p>We can then use <strong>plotClusterSpecificPeak</strong> to visualize the difference OCRs of these clusters/modules. To ensure the reproducibility of your results from running this step, we will save two files under the save path you provided, one is the specific grouping for each OCR (with the suffix <em>clustering_Cluster_Group.txt</em>). The other is the Zscore transformed matrix after clustering (suffix <em>clustering_Clustered_Scaled_Matr.tsv</em>).You can import this matrix and plot again without clustering.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="tissuesample-specific-analysis.html#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotClusterSpecificPeak</span>(<span class="at">specific_peak =</span> top1000, <span class="at">cluster_N =</span> <span class="dv">5</span>, <span class="at">save_path =</span> <span class="st">&quot;F:/cisDynet/example/&quot;</span>, <span class="at">file_prefix=</span><span class="st">&quot;Top1000_clustering&quot;</span>)</span></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/plotClusterSpecificPeak3-1.png" width="480" style="display: block; margin: auto;" /></p>
</div>
<div id="get-the-cluster-target-genes" class="section level2 hasAnchor" number="8.6">
<h2><span class="header-section-number">8.6</span> Get the cluster target genes<a href="tissuesample-specific-analysis.html#get-the-cluster-target-genes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Finally, for the specific OCR and grouping information we got in the previous step, we can use <strong>getClusterTargetGenes</strong> to see which target genes are in each cluster/module.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="tissuesample-specific-analysis.html#cb25-1" aria-hidden="true" tabindex="-1"></a>target <span class="ot">&lt;-</span> <span class="fu">getClusterTargetGenes</span>(<span class="at">cluster_group =</span> <span class="st">&quot;F:/cisDynet/example/Top1000_clustering_Cluster_Group.txt&quot;</span>)</span>
<span id="cb25-2"><a href="tissuesample-specific-analysis.html#cb25-2" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(target)</span></code></pre></div>
<pre><code>##   chrom     start       end peak_center  cluster       TSS            gene
## 1  chr1 100165637 100165976   100165806 Cluster4 100163798 ENSG00000223656
## 2  chr1 100194551 100194721   100194636 Cluster4 100163798 ENSG00000223656
## 3  chr1  10020974  10021652    10021313 Cluster2  10003465 ENSG00000162441
## 4  chr1 100223737 100224048   100223892 Cluster4 100232187 ENSG00000156869
## 5  chr1 100247222 100247415   100247318 Cluster3 100232187 ENSG00000156869
## 6  chr1 100458544 100458725   100458634 Cluster1 100435420 ENSG00000228084
##   strand distance
## 1      +    -2008
## 2      +   -30838
## 3      -    17848
## 4      -    -8295
## 5      -    15131
## 6      -    23214</code></pre>
</div>
<div id="go-enrichment" class="section level2 hasAnchor" number="8.7">
<h2><span class="header-section-number">8.7</span> GO Enrichment<a href="tissuesample-specific-analysis.html#go-enrichment" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We can simply run <strong>plotGO</strong> with the help of <em>clusterProfiler</em> to analyze the GO enrichment of genes for each cluster.</p>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="tissuesample-specific-analysis.html#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(org.Hs.eg.db)</span></code></pre></div>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="tissuesample-specific-analysis.html#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotGO</span>(<span class="at">target =</span> target,<span class="at">orgdb =</span> org.Hs.eg.db,<span class="at">N_top =</span> <span class="dv">3</span>)</span></code></pre></div>
<pre><code>## 2023-10-23 14:44:47 GO with Cluster4</code></pre>
<pre><code>## </code></pre>
<pre><code>## 2023-10-23 14:45:18 GO with Cluster2</code></pre>
<pre><code>## 2023-10-23 14:45:35 GO with Cluster3</code></pre>
<pre><code>## 2023-10-23 14:45:48 GO with Cluster1</code></pre>
<pre><code>## 2023-10-23 14:46:04 GO with Cluster5</code></pre>
<p><img src="bookdown-demo_files/figure-html/GO-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>In addition we can show the similarities and differences of the enriched pathways between different clusters in a network.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="tissuesample-specific-analysis.html#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotGONetwork</span>(<span class="at">target =</span> target,<span class="at">orgdb =</span> org.Hs.eg.db)</span></code></pre></div>
<pre><code>## 2023-10-23 14:46:11 Compare the GO results with clusters...</code></pre>
<p><img src="bookdown-demo_files/figure-html/GO%20Network-1.png" width="1344" style="display: block; margin: auto;" /></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="differential-ocr-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="time-course-data.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/06-Tissue-Specific-Analysis.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
