<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Tissue Specific Analysis | CAT: A toolkit for downstream analysis of bulk ATAC-seq data</title>
  <meta name="description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Tissue Specific Analysis | CAT: A toolkit for downstream analysis of bulk ATAC-seq data" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Tissue Specific Analysis | CAT: A toolkit for downstream analysis of bulk ATAC-seq data" />
  
  <meta name="twitter:description" content="A Chromatin Accessibility Toolkit for analysis ATAC-seq data (CAT)." />
  

<meta name="author" content="Tao Zhu" />


<meta name="date" content="2023-07-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="differential-ocr-analysis-two-samples.html"/>
<link rel="next" href="motif-enrichment.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Getting Started with CAT</a></li>
<li class="chapter" data-level="2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html"><i class="fa fa-check"></i><b>2</b> Snakemake Pipeline</a>
<ul>
<li class="chapter" data-level="2.1" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#input-data-format"><i class="fa fa-check"></i><b>2.1</b> Input data format</a></li>
<li class="chapter" data-level="2.2" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#paparing-your-config-file"><i class="fa fa-check"></i><b>2.2</b> Paparing your config file</a></li>
<li class="chapter" data-level="2.3" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#report-explainison"><i class="fa fa-check"></i><b>2.3</b> Report explainison</a></li>
<li class="chapter" data-level="2.4" data-path="snakemake-pipeline.html"><a href="snakemake-pipeline.html#data-quality-control-assement"><i class="fa fa-check"></i><b>2.4</b> Data Quality Control assement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="peak-annotaion.html"><a href="peak-annotaion.html"><i class="fa fa-check"></i><b>3</b> Peak annotaion</a>
<ul>
<li class="chapter" data-level="3.1" data-path="peak-annotaion.html"><a href="peak-annotaion.html#chipseeker-annotation"><i class="fa fa-check"></i><b>3.1</b> ChIPSeeker annotation</a></li>
<li class="chapter" data-level="3.2" data-path="peak-annotaion.html"><a href="peak-annotaion.html#peomoter-enhancer-annotaion"><i class="fa fa-check"></i><b>3.2</b> Peomoter/ Enhancer annotaion</a></li>
<li class="chapter" data-level="3.3" data-path="peak-annotaion.html"><a href="peak-annotaion.html#ocr-enrichemtn-with-bedtools-fisher"><i class="fa fa-check"></i><b>3.3</b> OCR Enrichemtn with bedtools fisher</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html"><i class="fa fa-check"></i><b>4</b> Peak Overlap Analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#venn-module-examples234-samples"><i class="fa fa-check"></i><b>4.1</b> Venn module examples(2,3,4 samples)</a></li>
<li class="chapter" data-level="4.2" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#upset-module-examples"><i class="fa fa-check"></i><b>4.2</b> UpSet module examples</a></li>
<li class="chapter" data-level="4.3" data-path="peak-overlap-analysis.html"><a href="peak-overlap-analysis.html#pairwise-module-examples"><i class="fa fa-check"></i><b>4.3</b> Pairwise module examples</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="differential-ocr-analysis-two-samples.html"><a href="differential-ocr-analysis-two-samples.html"><i class="fa fa-check"></i><b>5</b> Differential OCR Analysis (Two samples)</a>
<ul>
<li class="chapter" data-level="5.1" data-path="differential-ocr-analysis-two-samples.html"><a href="differential-ocr-analysis-two-samples.html#using-diffbind-to-identify-differentail-ocr"><i class="fa fa-check"></i><b>5.1</b> Using DiffBind to identify differentail OCR</a></li>
<li class="chapter" data-level="5.2" data-path="differential-ocr-analysis-two-samples.html"><a href="differential-ocr-analysis-two-samples.html#using-deseq2-to-identify-differentail-ocr"><i class="fa fa-check"></i><b>5.2</b> Using DEseq2 to identify differentail OCR</a></li>
<li class="chapter" data-level="5.3" data-path="differential-ocr-analysis-two-samples.html"><a href="differential-ocr-analysis-two-samples.html#plotting"><i class="fa fa-check"></i><b>5.3</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="tissue-specific-analysis.html"><a href="tissue-specific-analysis.html"><i class="fa fa-check"></i><b>6</b> Tissue Specific Analysis</a>
<ul>
<li class="chapter" data-level="6.1" data-path="tissue-specific-analysis.html"><a href="tissue-specific-analysis.html#multi-sample-analysis"><i class="fa fa-check"></i><b>6.1</b> Multi Sample Analysis</a></li>
<li class="chapter" data-level="6.2" data-path="tissue-specific-analysis.html"><a href="tissue-specific-analysis.html#time-series-analysis"><i class="fa fa-check"></i><b>6.2</b> Time-series Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="motif-enrichment.html"><a href="motif-enrichment.html"><i class="fa fa-check"></i><b>7</b> Motif Enrichment</a>
<ul>
<li class="chapter" data-level="7.1" data-path="motif-enrichment.html"><a href="motif-enrichment.html#fimo-scanning"><i class="fa fa-check"></i><b>7.1</b> FIMO Scanning</a></li>
<li class="chapter" data-level="7.2" data-path="motif-enrichment.html"><a href="motif-enrichment.html#simple-enrichment-analysissea"><i class="fa fa-check"></i><b>7.2</b> Simple Enrichment Analysis(SEA)</a></li>
<li class="chapter" data-level="7.3" data-path="motif-enrichment.html"><a href="motif-enrichment.html#gimmemotif"><i class="fa fa-check"></i><b>7.3</b> Gimmemotif</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="footprint-with-cat.html"><a href="footprint-with-cat.html"><i class="fa fa-check"></i><b>8</b> Footprint with CAT</a>
<ul>
<li class="chapter" data-level="8.1" data-path="footprint-with-cat.html"><a href="footprint-with-cat.html#footprint-with-tobias"><i class="fa fa-check"></i><b>8.1</b> Footprint with TOBIAS</a></li>
<li class="chapter" data-level="8.2" data-path="footprint-with-cat.html"><a href="footprint-with-cat.html#footprint-depth-calaulation"><i class="fa fa-check"></i><b>8.2</b> Footprint depth calaulation</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="co-accessibility-with-cat.html"><a href="co-accessibility-with-cat.html"><i class="fa fa-check"></i><b>9</b> Co-accessibility with CAT</a>
<ul>
<li class="chapter" data-level="9.1" data-path="co-accessibility-with-cat.html"><a href="co-accessibility-with-cat.html#data-input"><i class="fa fa-check"></i><b>9.1</b> Data Input</a></li>
<li class="chapter" data-level="9.2" data-path="co-accessibility-with-cat.html"><a href="co-accessibility-with-cat.html#co-accessibility-calaulation"><i class="fa fa-check"></i><b>9.2</b> Co-accessibility calaulation</a></li>
<li class="chapter" data-level="9.3" data-path="co-accessibility-with-cat.html"><a href="co-accessibility-with-cat.html#plotting-browser-tracks-of-co-accessibility"><i class="fa fa-check"></i><b>9.3</b> Plotting browser tracks of Co-accessibility</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html"><i class="fa fa-check"></i><b>10</b> Linking Peak to Gene</a>
<ul>
<li class="chapter" data-level="10.1" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#data-preparation"><i class="fa fa-check"></i><b>10.1</b> Data preparation</a></li>
<li class="chapter" data-level="10.2" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#link-ocr-to-genes"><i class="fa fa-check"></i><b>10.2</b> Link OCR to genes</a></li>
<li class="chapter" data-level="10.3" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#output-file"><i class="fa fa-check"></i><b>10.3</b> Output file</a></li>
<li class="chapter" data-level="10.4" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#plotting-1"><i class="fa fa-check"></i><b>10.4</b> Plotting</a></li>
<li class="chapter" data-level="10.5" data-path="linking-peak-to-gene.html"><a href="linking-peak-to-gene.html#target-gene-ontology-analysis"><i class="fa fa-check"></i><b>10.5</b> Target gene ontology analysis</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html"><i class="fa fa-check"></i><b>11</b> Building regulatory network</a>
<ul>
<li class="chapter" data-level="11.1" data-path="building-regulatory-network.html"><a href="building-regulatory-network.html#tobias-based"><i class="fa fa-check"></i><b>11.1</b> TOBIAS based</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="plotting-the-tracks-with-genome-browser.html"><a href="plotting-the-tracks-with-genome-browser.html"><i class="fa fa-check"></i><b>12</b> Plotting the tracks with genome browser</a>
<ul>
<li class="chapter" data-level="12.1" data-path="plotting-the-tracks-with-genome-browser.html"><a href="plotting-the-tracks-with-genome-browser.html#washu-epigenome-browser"><i class="fa fa-check"></i><b>12.1</b> WashU Epigenome Browser</a></li>
<li class="chapter" data-level="12.2" data-path="plotting-the-tracks-with-genome-browser.html"><a href="plotting-the-tracks-with-genome-browser.html#pygenometracks"><i class="fa fa-check"></i><b>12.2</b> pyGenomeTracks</a></li>
<li class="chapter" data-level="12.3" data-path="plotting-the-tracks-with-genome-browser.html"><a href="plotting-the-tracks-with-genome-browser.html#ggcoverage"><i class="fa fa-check"></i><b>12.3</b> ggcoverage</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="gwas-traits-snps-enrichment.html"><a href="gwas-traits-snps-enrichment.html"><i class="fa fa-check"></i><b>13</b> GWAS traits SNPs enrichment</a>
<ul>
<li class="chapter" data-level="13.1" data-path="gwas-traits-snps-enrichment.html"><a href="gwas-traits-snps-enrichment.html#using-cheers-to-do-the-enrichment"><i class="fa fa-check"></i><b>13.1</b> Using CHEERS to do the enrichment</a></li>
<li class="chapter" data-level="13.2" data-path="gwas-traits-snps-enrichment.html"><a href="gwas-traits-snps-enrichment.html#plotting-2"><i class="fa fa-check"></i><b>13.2</b> Plotting</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="integration-with-chip-seq.html"><a href="integration-with-chip-seq.html"><i class="fa fa-check"></i><b>14</b> Integration with ChIP-seq</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">CAT: A toolkit for downstream analysis of bulk ATAC-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="tissue-specific-analysis" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">Chapter 6</span> Tissue Specific Analysis<a href="tissue-specific-analysis.html#tissue-specific-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<pre><code></code></pre>
<div id="multi-sample-analysis" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> Multi Sample Analysis<a href="tissue-specific-analysis.html#multi-sample-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
</div>
<div id="time-series-analysis" class="section level2 hasAnchor" number="6.2">
<h2><span class="header-section-number">6.2</span> Time-series Analysis<a href="tissue-specific-analysis.html#time-series-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="differential-ocr-analysis-two-samples.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="motif-enrichment.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/05-Tissue-Specific-Analysis.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
