<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Function reference â€¢ cisDynet</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/yeti/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><!-- docsearch --><script src="../docsearch.js"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/docsearch.js/2.6.3/docsearch.min.css" integrity="sha256-QOSRU/ra9ActyXkIBbiIB144aDBdtvXBcNc3OTNuX/Q=" crossorigin="anonymous"><link href="../docsearch.css" rel="stylesheet"><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-158769145-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-158769145-1');
</script></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-index">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">cisDynet</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.0.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../index.html">
    <span class="fa fa-home"></span>
     
  </a>
</li>
<li>
  <a href="../book/index.html">
    <span class="fa fa-file-text-o"></span>
     
    Full Manual
  </a>
</li>
<li>
  <a href="../reference/index.html">
    <span class="fa fa-code"></span>
     
    Functions
  </a>
</li>
<li>
  <a href="../articles/Articles/faq.html">
    <span class="fa fa-question-circle-o"></span>
     
    FAQ
  </a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/tzhu-bio/cisDynet" class="external-link">
    <span class="fa fa-github"></span>
     
    cisDynet on GitHub
  </a>
</li>
      </ul><form class="navbar-form navbar-right hidden-xs hidden-sm" role="search">
        <div class="form-group">
          <input type="search" class="form-control" name="search-input" id="search-input" placeholder="Search..." aria-label="Search for..." autocomplete="off"></div>
      </form>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index"><colgroup><col class="alias"><col class="title"></colgroup><tbody><tr><th colspan="2">
          <h2 id="add-ers">Add-ers <a href="#add-ers" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Functions that add necessary annotation.</p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="addAnnotation.html">addAnnotation()</a></code> </p>
        </td>
        <td><p>A function to add the necessary annotation for **CAT**.</p></td>
      </tr><tr><td>
          <p><code><a href="addFootprint.html">addFootprint()</a></code> </p>
        </td>
        <td><p>A function to add necessary annotation for foorprinting analysis.</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="check-ers">Check-ers <a href="#check-ers" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Functions that check necessary annotation.</p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="checkFTAnno.html">checkFTAnno()</a></code> </p>
        </td>
        <td><p>Check for the presence of footprint annotation files.</p></td>
      </tr><tr><td>
          <p><code><a href="checkGeAnno.html">checkGeAnno()</a></code> </p>
        </td>
        <td><p>Check for the presence of genome annotation files.</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="cal-ers">Cal-ers <a href="#cal-ers" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Functions for calculating.</p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="getClusterNum.html">getClusterNum()</a></code> </p>
        </td>
        <td><p>A function to help you determine the cluster number before you use **plotClusterSpecificPeak**.</p></td>
      </tr><tr><td>
          <p><code><a href="getClusterTargetGenes.html">getClusterTargetGenes()</a></code> </p>
        </td>
        <td><p>A function to find the target genes based on specific peaks.</p></td>
      </tr><tr><td>
          <p><code><a href="getCoaccessible.html">getCoaccessible()</a></code> </p>
        </td>
        <td><p>Get the co-accessible peaks in genome wide.</p></td>
      </tr><tr><td>
          <p><code><a href="getCount.html">getCount()</a></code> </p>
        </td>
        <td><p>A function to get the count file for **chromVAR**.</p></td>
      </tr><tr><td>
          <p><code><a href="getCountSplit.html">getCountSplit()</a></code> </p>
        </td>
        <td><p>A function to get the count file for **GWASNorm**.</p></td>
      </tr><tr><td>
          <p><code><a href="getDiffPeak.html">getDiffPeak()</a></code> </p>
        </td>
        <td><p>A function to identify the differential peaks between two groups.</p></td>
      </tr><tr><td>
          <p><code><a href="getDiffTargetGenes.html">getDiffTargetGenes()</a></code> </p>
        </td>
        <td><p>A function to call the differential peak target genes.</p></td>
      </tr><tr><td>
          <p><code><a href="getFootprintScore.html">getFootprintScore()</a></code> </p>
        </td>
        <td><p>A function to get the footprint score.</p></td>
      </tr><tr><td>
          <p><code><a href="getGeneBed.html">getGeneBed()</a></code> </p>
        </td>
        <td><p>Get the specific gene coordinates and flanks.</p></td>
      </tr><tr><td>
          <p><code><a href="getGER.html">getGER()</a></code> </p>
        </td>
        <td><p>The function to integrate the GWAS enrichment results.</p></td>
      </tr><tr><td>
          <p><code><a href="getJasparMotifs2018.html">getJasparMotifs2018()</a></code> </p>
        </td>
        <td><p>Get the motifs from JASPAR database (2018).</p></td>
      </tr><tr><td>
          <p><code><a href="getMotif2Gene.html">getMotif2Gene()</a></code> </p>
        </td>
        <td><p>Convert motif ID to gene ID.</p></td>
      </tr><tr><td>
          <p><code><a href="getNetwork.html">getNetwork()</a></code> </p>
        </td>
        <td><p>Get the regulatory network.</p></td>
      </tr><tr><td>
          <p><code><a href="getPeak2Gene.html">getPeak2Gene()</a></code> </p>
        </td>
        <td><p>Calculate the ATAC-to-RNA / Peak-to-Gene links.</p></td>
      </tr><tr><td>
          <p><code><a href="getRankMotif.html">getRankMotif()</a></code> </p>
        </td>
        <td><p>Rank the motif deviation score.</p></td>
      </tr><tr><td>
          <p><code><a href="getSNP2Gene.html">getSNP2Gene()</a></code> </p>
        </td>
        <td><p>A function help us look for which genes these SNPs might affect the expression of.</p></td>
      </tr><tr><td>
          <p><code><a href="getSpecificPeak.html">getSpecificPeak()</a></code> </p>
        </td>
        <td><p>A function to identify sample specific peaks.</p></td>
      </tr><tr><td>
          <p><code><a href="getSuperEnhancer.html">getSuperEnhancer()</a></code> </p>
        </td>
        <td><p>A function to identify the super enahncers.</p></td>
      </tr><tr><td>
          <p><code><a href="getTimeATAC.html">getTimeATAC()</a></code> </p>
        </td>
        <td><p>Fitting the ATAC signal along the time.</p></td>
      </tr><tr><td>
          <p><code><a href="getTimeGene.html">getTimeGene()</a></code> </p>
        </td>
        <td><p>Get the fitting time genes.</p></td>
      </tr><tr><td>
          <p><code><a href="getTimeRNA.html">getTimeRNA()</a></code> </p>
        </td>
        <td><p>Plot the RNA dynamic along the pseudo time.</p></td>
      </tr><tr><td>
          <p><code><a href="getTopSpecifcPeaks.html">getTopSpecifcPeaks()</a></code> </p>
        </td>
        <td><p>Get the top N specific peaks in each sample.</p></td>
      </tr><tr><td>
          <p><code><a href="getTriads.html">getTriads()</a></code> </p>
        </td>
        <td><p>A function to compare the chromatin accessibility across thress samples.</p></td>
      </tr><tr><td>
          <p><code><a href="getTriadsTargetGenes.html">getTriadsTargetGenes()</a></code> </p>
        </td>
        <td><p>A function to get the target genes of triads peak.</p></td>
      </tr><tr><td>
          <p><code><a href="get_beds.html">get_beds()</a></code> </p>
        </td>
        <td><p>Read beds into list, can be arbitary dir_path.</p></td>
      </tr><tr><td>
          <p><code><a href="get_combine_result.html">get_combine_result()</a></code> </p>
        </td>
        <td><p>Get intersection result which can be used as input of venn/upset plot.</p></td>
      </tr><tr><td>
          <p><code><a href="get_intersect.html">get_intersect()</a></code> </p>
        </td>
        <td><p>Update ident of bed_list based on intersection result</p></td>
      </tr><tr><td>
          <p><code><a href="get_overlap.html">get_overlap()</a></code> </p>
        </td>
        <td><p>Calculate the overlap interval.</p></td>
      </tr><tr><td>
          <p><code><a href="get_singlebed.html">get_singlebed()</a></code> </p>
        </td>
        <td><p>Read beds into a single tibble.</p></td>
      </tr><tr><td>
          <p><code><a href="get_unique.html">get_unique()</a></code> </p>
        </td>
        <td><p>Get distinctive records of A.</p></td>
      </tr><tr><td>
          <p><code><a href="get_upset_input.html">get_upset_input()</a></code> </p>
        </td>
        <td><p>Get expression input which can be used in upset plot</p></td>
      </tr><tr><td>
          <p><code><a href="get_venn_key.html">get_venn_key()</a></code> </p>
        </td>
        <td><p>Get keys which can be used in venn plot.</p></td>
      </tr><tr><td>
          <p><code><a href="annoMergedPeaks.html">annoMergedPeaks()</a></code> </p>
        </td>
        <td><p>Determining the merged peak category (Proximal/Distal/Intragenic).
Determining the merged peak category (Proximal / Distal / Intragenic).</p></td>
      </tr><tr><td>
          <p><code><a href="annoPeaks.html">annoPeaks()</a></code> </p>
        </td>
        <td><p>Annotating the peaks with ChIPSeeker.</p></td>
      </tr><tr><td>
          <p><code><a href="calc_dist_matrix.html">calc_dist_matrix()</a></code> </p>
        </td>
        <td><p>Calculate distance for matrix.</p></td>
      </tr><tr><td>
          <p><code><a href="calFisherPeaks.html">calFisherPeaks()</a></code> </p>
        </td>
        <td><p>A function to calculate the two group peaks enrichment based on Fisher Test.</p></td>
      </tr><tr><td>
          <p><code><a href="calJSD.html">calJSD()</a></code> </p>
        </td>
        <td><p>A function to calculate the JSD score.</p></td>
      </tr><tr><td>
          <p><code><a href="calPairwise.html">calPairwise()</a></code> </p>
        </td>
        <td><p>Calculate pairwised intersection of bed files.</p></td>
      </tr><tr><td>
          <p><code><a href="calSPM.html">calSPM()</a></code> </p>
        </td>
        <td><p>A function to quantify tissue-specificity as the Specificity Measure (SPM).</p></td>
      </tr><tr><td>
          <p><code><a href="compareFootprints.html">compareFootprints()</a></code> </p>
        </td>
        <td><p>A function to compare the Tn5 signal around the motif in the whole genome.</p></td>
      </tr><tr><td>
          <p><code><a href="GWASEnrichment.html">GWASEnrichment()</a></code> </p>
        </td>
        <td><p>Do enrichment analysis with GWAS variants.</p></td>
      </tr><tr><td>
          <p><code><a href="GWASNorm.html">GWASNorm()</a></code> </p>
        </td>
        <td><p>Preprocessing the data for GWAS enrichment.</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="plotting">Plotting <a href="#plotting" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Functions for plotting in cisDynet.</p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="plotClusterSpecificPeak.html">plotClusterSpecificPeak()</a></code> </p>
        </td>
        <td><p>A function to help you plot the sample specific peaks with heatmap.</p></td>
      </tr><tr><td>
          <p><code><a href="plotCoGeneTracks.html">plotCoGeneTracks()</a></code> </p>
        </td>
        <td><p>Plot specific gene ATAC signal and co-accessible with tracks.</p></td>
      </tr><tr><td>
          <p><code><a href="plotCoTracks.html">plotCoTracks()</a></code> </p>
        </td>
        <td><p>Plot genome tracks with co-accessible information.</p></td>
      </tr><tr><td>
          <p><code><a href="plotDendrogram.html">plotDendrogram()</a></code> </p>
        </td>
        <td><p>A function used to plot tree clustering result.</p></td>
      </tr><tr><td>
          <p><code><a href="plotDiffFootprint.html">plotDiffFootprint()</a></code> </p>
        </td>
        <td><p>Plot the differential footprint with Volcano, Bar or Lollipop.</p></td>
      </tr><tr><td>
          <p><code><a href="plotEnrich.html">plotEnrich()</a></code> </p>
        </td>
        <td><p>A function to plot the enrichment rsult based on **calFisherPeaks**.</p></td>
      </tr><tr><td>
          <p><code><a href="plotFlower.html">plotFlower()</a></code> </p>
        </td>
        <td><p>Draw flower plot which represents the number of merged intervals.</p></td>
      </tr><tr><td>
          <p><code><a href="plotFootprintScore.html">plotFootprintScore()</a></code> </p>
        </td>
        <td><p>A function to plot all samples footprint with scatter / heatmap.</p></td>
      </tr><tr><td>
          <p><code><a href="plotFootprintScoreExp.html">plotFootprintScoreExp()</a></code> </p>
        </td>
        <td><p>Plot the correlation of footprint score and TF expression.</p></td>
      </tr><tr><td>
          <p><code><a href="plotFragments.html">plotFragments()</a></code> </p>
        </td>
        <td><p>A function to plot the insertion fragment size.</p></td>
      </tr><tr><td>
          <p><code><a href="plotGeneTracks.html">plotGeneTracks()</a></code> </p>
        </td>
        <td><p>Plot specific gene ATAC signal with tracks.</p></td>
      </tr><tr><td>
          <p><code><a href="plotGER.html">plotGER()</a></code> </p>
        </td>
        <td><p>A function to plot the GWAS Enrichment Results.</p></td>
      </tr><tr><td>
          <p><code><a href="plotGO.html">plotGO()</a></code> </p>
        </td>
        <td><p>A function to plot the every cluster target genes GO enrichment.</p></td>
      </tr><tr><td>
          <p><code><a href="plotGONetwork.html">plotGONetwork()</a></code> </p>
        </td>
        <td><p>A function to compare the GO enrichment result with Network.</p></td>
      </tr><tr><td>
          <p><code><a href="plotMA.html">plotMA()</a></code> </p>
        </td>
        <td><p>A fucntion to plot MA plot</p></td>
      </tr><tr><td>
          <p><code><a href="plotNetwork.html">plotNetwork()</a></code> </p>
        </td>
        <td><p>Plot the regulatory network.</p></td>
      </tr><tr><td>
          <p><code><a href="plotP2GHeatmap.html">plotP2GHeatmap()</a></code> </p>
        </td>
        <td><p>Plot all the Peak2Gene links with heatmap.</p></td>
      </tr><tr><td>
          <p><code><a href="plotP2GTracks.html">plotP2GTracks()</a></code> </p>
        </td>
        <td><p>Plot specific gene with tracks and peak2gene linkages.</p></td>
      </tr><tr><td>
          <p><code><a href="plotPCA.html">plotPCA()</a></code> </p>
        </td>
        <td><p>Plot the PCA plot based on quantification.</p></td>
      </tr><tr><td>
          <p><code><a href="plotPDI.html">plotPDI()</a></code> </p>
        </td>
        <td><p>Determining the peak category (Proximal/Distal/Intragenic).</p></td>
      </tr><tr><td>
          <p><code><a href="plotPeakGene.html">plotPeakGene()</a></code> </p>
        </td>
        <td><p>Plot one gene / one peak scatter plot.</p></td>
      </tr><tr><td>
          <p><code><a href="plotSpecificPeak.html">plotSpecificPeak()</a></code> </p>
        </td>
        <td><p>Plot the sample specific peaks.</p></td>
      </tr><tr><td>
          <p><code><a href="plotSummitDis.html">plotSummitDis()</a></code> </p>
        </td>
        <td><p>Plot the distance of peak summit to TSS.</p></td>
      </tr><tr><td>
          <p><code><a href="plotTimeAll.html">plotTimeAll()</a></code> </p>
        </td>
        <td><p>Plot the fitting time ATAC and RNA.</p></td>
      </tr><tr><td>
          <p><code><a href="plotTimeGene.html">plotTimeGene()</a></code> </p>
        </td>
        <td><p>Plot the specific gene expression along the time.</p></td>
      </tr><tr><td>
          <p><code><a href="plotTracks.html">plotTracks()</a></code> </p>
        </td>
        <td><p>Plot the genome signal with tracks.</p></td>
      </tr><tr><td>
          <p><code><a href="plotTriads.html">plotTriads()</a></code> </p>
        </td>
        <td><p>A function to plot the triads result.</p></td>
      </tr><tr><td>
          <p><code><a href="plotTSS.html">plotTSS()</a></code> </p>
        </td>
        <td><p>A function to plot the Tn5 signal around the TSSs.</p></td>
      </tr><tr><td>
          <p><code><a href="plotUpset.html">plotUpset()</a></code> </p>
        </td>
        <td><p>Draw upset plot of bed intersection.</p></td>
      </tr><tr><td>
          <p><code><a href="plotVenn.html">plotVenn()</a></code> </p>
        </td>
        <td><p>Draw venn plot of bed intersection.</p></td>
      </tr><tr><td>
          <p><code><a href="plotVolcano.html">plotVolcano()</a></code> </p>
        </td>
        <td><p>A function to plot the differential result.</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="others">Others <a href="#others" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Other functions in cisDynet.</p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="logfile.html">logfile()</a></code> </p>
        </td>
        <td><p>Message with a timestamp.</p></td>
      </tr><tr><td>
          <p><code><a href="orderMatrix.html">orderMatrix()</a></code> </p>
        </td>
        <td><p>Order the rows along with pseudo time.</p></td>
      </tr><tr><td>
          <p><code><a href="generate_windows.html">generate_windows()</a></code> </p>
        </td>
        <td><p>Generate window for a given bed file.</p></td>
      </tr><tr><td>
          <p><code><a href="makeSummarizedExperiment.html">makeSummarizedExperiment()</a></code> </p>
        </td>
        <td><p>Make SummarizedExperiment for chromVAR input.</p></td>
      </tr><tr><td>
          <p><code><a href="parse_bedpath.html">parse_bedpath()</a></code> </p>
        </td>
        <td><p>Parse bedpaths as file based path.</p></td>
      </tr><tr><td>
          <p><code><a href="quantile_normalization.html">quantile_normalization()</a></code> </p>
        </td>
        <td><p>Making quantile normalization for matrix.</p></td>
      </tr><tr><td>
          <p><code><a href="rowZscores.html">rowZscores()</a></code> </p>
        </td>
        <td><p>Scale matrix with Z-score.</p></td>
      </tr><tr><td>
          <p><code><a href="change_res_ident.html">change_res_ident()</a></code> </p>
        </td>
        <td><p>Change identity of the beds$rec based on intersection result.</p></td>
      </tr><tr><td>
          <p><code><a href="flower_plot.html">flower_plot()</a></code> </p>
        </td>
        <td><p>Draw flower plot.</p></td>
      </tr><tr><td>
          <p><code><a href="init_result_df.html">init_result_df()</a></code> </p>
        </td>
        <td><p>Init dataframe which would be used in intersection results based on bed ident.</p></td>
      </tr><tr><td>
          <p><code><a href="loadBigWig.html">loadBigWig()</a></code> <code><a href="loadBigWig.html">loadFullBigWig()</a></code> </p>
        </td>
        <td><p>loadBigWig</p></td>
      </tr></tbody></table></div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Tao Zhu.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer></div>

  
<script src="https://cdnjs.cloudflare.com/ajax/libs/docsearch.js/2.6.1/docsearch.min.js" integrity="sha256-GKvGqXDznoRYHCwKXGnuchvKSwmx9SRMrZOTh2g4Sb0=" crossorigin="anonymous"></script><script>
  docsearch({
    
    
    apiKey: '252c2c36fedc32b3ed803d0db4877b96',
    indexName: 'cisDynet',
    inputSelector: 'input#search-input.form-control',
    transformData: function(hits) {
      return hits.map(function (hit) {
        hit.url = updateHitURL(hit);
        return hit;
      });
    }
  });
</script></body></html>

